<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title>
            Family of four gets their genomes sequenced
        </title>
        <meta name="viewport" content="width=device-width,initial-scale=1.0">
        <link rel="stylesheet" href="/static/css/main.min.css" type="text/css" media="all"><!--[if lte IE 8]>
            <link rel="stylesheet" href="/static/css/ie-fixes.css" type="text/css" media="all">
            <script src="/static/js/html5shiv.js"></script>
        <![endif]-->
        <meta http-equiv="last-modified" content="Fri, 12 Mar 2010 03:28:00 GMT">
        <link rel="alternate" type="application/atom+xml" title="All Posts (Atom)" href="/feeds/all.atom">
        <link rel="alternate" type="application/rss+xml" title="All Posts (RSS)" href="/feeds/all.rss">
        <link rel="icon" type="image/jpeg" sizes="287x287" href="/static/img/favicon-287x287.jpg">
        <link rel="icon" type="image/jpeg" sizes="128x128" href="/static/img/favicon-128x128.jpg">
        <script type="application/javascript">
var _prum={id:"5166be01e6e53d1007000001"};var PRUM_EPISODES=PRUM_EPISODES||{};PRUM_EPISODES.q=[];PRUM_EPISODES.mark=function(b,a){PRUM_EPISODES.q.push(["mark",b,a||new Date().getTime()])};PRUM_EPISODES.measure=function(b,a,b){PRUM_EPISODES.q.push(["measure",b,a,b||new Date().getTime()])};PRUM_EPISODES.done=function(a){PRUM_EPISODES.q.push(["done",a])};PRUM_EPISODES.mark("firstbyte");(function(){var b=document.getElementsByTagName("script")[0];var a=document.createElement("script");a.type="text/javascript";a.async=true;a.charset="UTF-8";a.src="//rum-static.pingdom.net/prum.min.js";b.parentNode.insertBefore(a,b)})();
        </script>
    </head>
    <body class="blog post_detail" itemscope itemtype="http://schema.org/BlogPosting">
        <nav id="site-nav">
            <header id="about">
                <h1>
                    <a href="/about/">Chris Adams</a>
                </h1>
                <h2>
                    Programmer, cyclist, photographer
                </h2>
            </header>
            <ul class="links">
                <li>
                    <a href="/">Home</a>
                </li>
                <li>
                    <a href="/about/">About</a>
                </li>
                <li>
                    <a href="/feeds/all.atom" title="Site Feed">Site Feed</a>
                </li>
            </ul>
            <ul class="social-networks">
                <li>
                    <a href="https://github.com/acdha" rel="me">Github</a>
                </li>
                <li>
                    <a href="http://bitbucket.org/acdha" rel="me">Bitbucket</a>
                </li>
                <li>
                    <a href="http://delicious.com/acdha" rel="me">del.icio.us</a>
                </li>
                <li>
                    <a href="http://twitter.com/acdha" rel="me">Twitter</a>
                </li>
                <li>
                    <a href="https://plus.google.com/116562742092842686896?rel=author" rel="me">Google+</a>
                </li>
                <li>
                    <a href="http://www.flickr.com/photos/acdha" rel="me">Flickr</a>
                </li>
                <li>
                    <a href="http://www.linkedin.com/in/acdha" rel="me">LinkedIn</a>
                </li>
                <li>
                    <a href="http://connect.garmin.com/explore?owner=acdha" rel="me">Garmin Connect</a>
                </li>
            </ul>
            <div id="site-search"></div>
        </nav>
        <section id="main">
            <article class="post">
                <header>
                    <meta itemprop="dateCreated" content="2010-03-11T22:28:00-04:00">
                    <meta itemprop="dateModified" content="2010-03-11T22:28:00-04:00"><time class="date" itemprop="datePublished" datetime="2010-03-12T02:28:00+00:00">Mar 12</time>
                    <h2 itemprop="title">
                        Family of four gets their genomes sequenced
                    </h2>
                </header>
                <div class="body" itemprop="articleBody">
                    <div class="googlereader description" data-google-id="tag:google.com,2005:reader/item/d3ab1c58ca2a9e23">
                        <blockquote>
                            <a href="http://arstechnica.com/science/news/2010/03/family-of-four-gets-their-genomes-sequenced.ars?utm_source=rss&amp;utm_medium=rss&amp;utm_campaign=rss"><img vspace="4" hspace="4" border="0" align="right" src="http://static.arstechnica.com/assets/2009/11/dna_green_ars-thumb-230x130-9526-f.jpg"></a>
                            <p>
                                Late last year, <a href="http://arstechnica.com/science/news/2009/11/complete-genomics-produces-a-cheapwell-5000human-genome.ars">we described a genome sequencing technique</a> that brought the price of consumables down to under $5,000. That technique, offered by Complete Genomics, has now been put to use: all the genomes have been obtained from a family of four in which both children suffer from two genetic disorders. In addition to identifying likely causative mutations, the full family pedigree has produced new measures of human mutation and recombination.
                            </p>
                            <p>
                                So far, as each genome has been completed, it's typically been compared to a reference genome that's meant to represent a "typical" human. But the human population is large and diverse, and the differences between a typical person and the reference may have been present in our population for thousands of years. In contrast, by knowing the sequence of a child and both its parents, the changes in DNA that occur as a result of recombinations and mutations in each parent's germ cells can be tracked in exquisite detail.
                            </p>
                            <p>
                                Of course, you first have to get rid of the errors. Any method of sequencing DNA has a known error rate, and there are certain sequences in the genome that are more prone to these mistakes than others.
                            </p>
                            <p>
                                All together, sequencing the genome identified over 4 million bases in which at least one of the genomes differed from the reference sequence. But nearly a million of these were identical in all four individuals; another 3.4 million had been identified as sites of common variations within the human genome. When all of these were eliminated, there were only 323,255 base changes that appeared to be distinct to this family. The authors of the paper then focused on getting rid of some of the errors.
                            </p>
                            <p>
                                Some of these errors are caused by repetitive sequences, which sometimes cause genome assembly algorithms to delete portions of the repeat. That dropped the number down to about 50,000 differences that were either new mutations or sequencing errors. They resequenced every one of these regions and found only 28 that appeared to be new mutations in the offspring; each of these was directly confirmed by mass spectroscopy of the relevant stretch of DNA. The authors estimate that, all told, they've eliminated approximately 70 percent of the sequencing errors, producing an accuracy of 99.999 percent.
                            </p>
                            <p>
                                After estimating the false negative rate, the authors concluded that humans have a mutation rate of 1.1x10<sup><small>-8</small></sup> at each base, which means that every individual is likely to have been born with approximately 70 new mutations. That's a bit less than half of previous estimates, but it's within the range defined by our differences with the chimp genome and an estimated time of divergence of 5 million years.
                            </p>
                            <p>
                                The precise map of differences also enabled the authors to track where pieces of the original parental genomes had been swapped by recombination. The precise bases can't be identified, given that most of the genome is identical in all four individuals, but they were able to identify 155 crossover sites within a median precision of 2,600 bases. Most of these occurred within known "hotspots" of recombination that had been identified previously.
                            </p>
                            <p>
                                As if all of this data wasn't enough, the family itself had been chosen because both children (and neither parent) suffers from two genetic diseases: <a href="http://www.ncbi.nlm.nih.gov/entrez/dispomim.cgi?id=263750">Miller syndrome</a> and <a href="http://www.ncbi.nlm.nih.gov/sites/entrez">primary ciliary dyskinesia</a>. The former has not had a gene definitively associated with it; the latter has had a number.
                            </p>
                            <p>
                                The simplest explanation of this would be a single recessive mutation, with the parents heterozygous, and the offspring homozygous. Since the diseases are rare, the authors excluded any base differences that have been previously identified as common within the human population. Only a single gene fit this pattern when identical mutations were considered. But it's possible for different mutations in the same gene to cause a single phenotype, with each parent carrying a distinct base change. Three additional genes matched this pattern.
                            </p>
                            <p>
                                So the authors sequenced these genes in two other individuals with Miller's syndrome, and identified <em>DHODH</em>, a gene previously suggested to be associated with the disease as its likely cause. One surprise is that the primary ciliary dyskinesia is likely to be caused by a completely separate mutation. <em>DNAH5</em>, another of the four genes to come through this analysis, had previously been identified as a cause of that disorder. So, the family appears to be unlucky enough to be dealing with two rare, recessive mutations.
                            </p>
                            <p>
                                That we've reached the point where this work is even possible is simply amazing. I've really got no words for the fact that it was done by a mere 15-author collaboration, only two of whom hail from Complete Genomics. As the authors point out, we're at the point where it may be cheaper and easier to sequence entire pedigrees than hunt down enough affected individuals to identify a Mendelian trait by traditional methods.
                            </p>
                            <p>
                                The fact that the approach generates additional useful data—things like human mutation rates and recombination locations have always been based on much rougher estimates—is really quite a significant bonus. I anticipate that this won't be the first paper of its sort, and these estimates will continue to be refined as more family pedigrees are available at the genome level.
                            </p>
                            <p>
                                <em>Science</em>, 2010. DOI: <a href="http://dx.doi.org/10.1126/science.1186802">10.1126/science.1186802</a> &nbsp;(<a href="http://arstechnica.com/science/news/2010/03/dois-and-their-discontents-1.ars">About DOIs</a>).
                            </p>
                            <p>
                                <a href="http://arstechnica.com/science/news/2010/03/family-of-four-gets-their-genomes-sequenced.ars?comments=1&amp;utm_source=rss&amp;utm_medium=rss&amp;utm_campaign=rss#comments-bar">Read the comments on this post</a>
                            </p>
                        </blockquote>
                    </div>
                    <p class="bookmark-source">
                        Source: <a href="http://arstechnica.com/science/news/2010/03/family-of-four-gets-their-genomes-sequenced.ars?utm_source=rss&amp;utm_medium=rss&amp;utm_campaign=rss">http://arstechnica.com/science/news/2010/03/family-of-four-gets-their-genomes-sequenced.ars?utm_source=rss&amp;utm_medium=rss&amp;utm_campaign=rss</a>
                    </p>
                </div>
            </article>
            <nav id="post-nav"></nav><script id="discus-javascript">
    var disqus_shortname = 'improbable';

                (function() {
                    var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
                    dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
                    document.getElementsByTagName('body')[0].appendChild(dsq);
                })();
            </script><a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>
            <div id="disqus_thread"></div>
        </section>
        <footer id="site-footer">
            <p>
                This site is purely my personal work and does not reflect the views of my employer.
            </p>
            <p class="license">
                <a class="icon" rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US"><img alt="Creative Commons License" style="border-width:0" src="http://i.creativecommons.org/l/by-sa/3.0/88x31.png"></a> This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US">Creative Commons Attribution-ShareAlike 3.0 Unported License</a>.
            </p>
        </footer><script async="" defer src="/static/js/common.js">
</script><script id="google-analytics">
    var _gaq = _gaq || [];
            _gaq.push(['_setAccount', 'UA-2097834-1']);
            _gaq.push(['_setDomainName', 'improbable.org']);
            _gaq.push(['_trackPageview']);

            (function() {
                var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
                ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
                var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
            })();
        </script>
    </body>
</html>
