<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title>
            Genome hints humans, Neanderthals rolled in prehistoric hay
        </title>
        <meta name="viewport" content="width=device-width,initial-scale=1.0">
        <link rel="stylesheet" href="/static/css/main.min.css" type="text/css" media="all"><!--[if lte IE 8]>
            <link rel="stylesheet" href="/static/css/ie-fixes.css" type="text/css" media="all">
            <script src="/static/js/html5shiv.js"></script>
        <![endif]-->
        <meta http-equiv="last-modified" content="Thu, 06 May 2010 18:13:00 GMT">
        <link rel="alternate" type="application/atom+xml" title="All Posts (Atom)" href="/feeds/all.atom">
        <link rel="alternate" type="application/rss+xml" title="All Posts (RSS)" href="/feeds/all.rss">
        <link rel="icon" type="image/jpeg" sizes="287x287" href="/static/img/favicon-287x287.jpg">
        <link rel="icon" type="image/jpeg" sizes="128x128" href="/static/img/favicon-128x128.jpg">
        <script type="application/javascript">
var _prum={id:"5166be01e6e53d1007000001"};var PRUM_EPISODES=PRUM_EPISODES||{};PRUM_EPISODES.q=[];PRUM_EPISODES.mark=function(b,a){PRUM_EPISODES.q.push(["mark",b,a||new Date().getTime()])};PRUM_EPISODES.measure=function(b,a,b){PRUM_EPISODES.q.push(["measure",b,a,b||new Date().getTime()])};PRUM_EPISODES.done=function(a){PRUM_EPISODES.q.push(["done",a])};PRUM_EPISODES.mark("firstbyte");(function(){var b=document.getElementsByTagName("script")[0];var a=document.createElement("script");a.type="text/javascript";a.async=true;a.charset="UTF-8";a.src="//rum-static.pingdom.net/prum.min.js";b.parentNode.insertBefore(a,b)})();
        </script>
    </head>
    <body class="blog post_detail" itemscope itemtype="http://schema.org/BlogPosting">
        <nav id="site-nav">
            <header id="about">
                <h1>
                    <a href="/about/">Chris Adams</a>
                </h1>
                <h2>
                    Programmer, cyclist, photographer
                </h2>
            </header>
            <ul class="links">
                <li>
                    <a href="/">Home</a>
                </li>
                <li>
                    <a href="/about/">About</a>
                </li>
                <li>
                    <a href="/feeds/all.atom" title="Site Feed">Site Feed</a>
                </li>
            </ul>
            <ul class="social-networks">
                <li>
                    <a href="https://github.com/acdha" rel="me">Github</a>
                </li>
                <li>
                    <a href="http://bitbucket.org/acdha" rel="me">Bitbucket</a>
                </li>
                <li>
                    <a href="http://delicious.com/acdha" rel="me">del.icio.us</a>
                </li>
                <li>
                    <a href="http://twitter.com/acdha" rel="me">Twitter</a>
                </li>
                <li>
                    <a href="https://plus.google.com/116562742092842686896?rel=author" rel="me">Google+</a>
                </li>
                <li>
                    <a href="http://www.flickr.com/photos/acdha" rel="me">Flickr</a>
                </li>
                <li>
                    <a href="http://www.linkedin.com/in/acdha" rel="me">LinkedIn</a>
                </li>
                <li>
                    <a href="http://connect.garmin.com/explore?owner=acdha" rel="me">Garmin Connect</a>
                </li>
            </ul>
            <div id="site-search"></div>
        </nav>
        <section id="main">
            <article class="post">
                <header>
                    <meta itemprop="dateCreated" content="2010-05-06T14:13:00-04:00">
                    <meta itemprop="dateModified" content="2010-05-06T14:13:00-04:00"><time class="date" itemprop="datePublished" datetime="2010-05-06T18:13:00+00:00">May 06</time>
                    <h2 itemprop="title">
                        Genome hints humans, Neanderthals rolled in prehistoric hay
                    </h2>
                </header>
                <div class="body" itemprop="articleBody">
                    <div class="googlereader description" data-google-id="tag:google.com,2005:reader/item/5e6f57cbcac43ec5">
                        <blockquote>
                            <a href="http://arstechnica.com/science/news/2010/05/neanderthal-genome-draft-done-indicates-interbreeding-occurred.ars?utm_source=rss&amp;utm_medium=rss&amp;utm_campaign=rss"><img vspace="4" hspace="4" border="0" align="right" width="230" height="129" src="http://static.arstechnica.com/assets/2009/02/neanderthal-thumb-230x130-2197-f.jpg"></a>
                            <p>
                                This news has been in the works for several years, from the <a href="http://arstechnica.com/science/news/2006/11/6005.ars">first reports</a> of <a href="http://arstechnica.com/science/news/2006/05/3992.ars">gene sequences</a> in 2006 to a <a href="http://arstechnica.com/science/news/2009/02/a-neanderthal-in-the-family-ars-at-aaas.ars">public overview</a> at last year's AAAS meeting. Today's issue of <em>Science</em> now contains the final product: the first draft sequence of one of our recent human relatives, the Neanderthal. &nbsp;
                            </p>
                            <p>
                                There's lots of information here for the genomics obsessive, which we'll get to later, but there's also some generally significant findings, including evidence that the ancestors of some human populations interbred with Neanderthals.
                            </p>
                            <p>
                                The DNA used for the sequencing comes primarily from three bone fragments—described as being "of little morphological value"—from a cave in Croatia. Even so, the DNA was extracted from bone powder produced with a small dental drill. The authors suggest that, as long as minor damage can be tolerated, it's worth trying to get DNA out of bones.
                            </p>
                            <p>
                                The paper confirms <a href="http://arstechnica.com/science/news/2007/10/human-sequences-cropping-up-in-neanderthal-genome.ars">earlier reports</a> that initial sequences were contaminated with DNA from modern humans, and describes the extensive quality controls that were put in place to limit the new data to Neanderthal.
                            </p>
                            <p>
                                Various measures indicate that the new sequence that's being released is over 99 percent Neanderthal. In total, it's about 5.3 Gigabases worth of sequence, meaning that every base in the Neanderthal genome was sequenced, on average, 1.3 times. Obviously, the distribution is not that even, and many areas remain unsequenced.
                            </p>
                            <p>
                                What we do have confirms that the three bones come from distinct individuals, although two may have been related, as their mitochondrial DNA sequences were identical. The primary samples were supplemented with additional sequences from bones found in Spain, the Caucasus, and the initial skeleton from the Neander valley.&nbsp;
                            </p>
                            <p>
                                The similarity of the sequences across all these samples indicates that the species was about equally distinct from humans across most of its geographic range. For the most part, Neanderthals looked like humans. Ninety percent of the time that there was a difference between human and chimp DNA sequences, the human variant showed up in Neanderthals.
                            </p>
                            <p>
                                What about the differences? When it comes to protein coding genes, they're pretty minor. Only 78 differences in the sequences that encode proteins are uniformly present in humans but absent in Neanderthals. Only five of those would change the primary structure of the protein.&nbsp;
                            </p>
                            <p>
                                The similarity is even more striking when you look at copy number variations, which can alter the dose of genes. Neanderthals have lots of CNVs, but all but three of them show up in other primates, and the three exceptions don't contain any genes.
                            </p>
                            <p>
                                Much of the action instead seems to be happening in areas that may regulate the expression of genes. There were over 230 changes apparent in the parts of genes that flank the protein-coding section (the 5' and 3' UTRs). In the areas that have been identified as Human Accelerated Regions (HARs) based on the large differences between humans and chimps, the Neanderthals had the human form 90 percent of the time, but that still left 45 HARs in which humans have picked up significant differences since they diverged from Neanderthals.
                            </p>
                            <h3>
                                Sweeping genes and interbreeding
                            </h3>
                            <p>
                                The authors also went looking for cases where there was evidence of what's termed a <a href="http://arstechnica.com/science/news/2006/11/5909.ars">selective sweep</a>, where a useful mutation occurs and spreads through the population, dragging its area of the chromosome along with it. We can detect these by looking for large chunks of the chromosome that are essentially identical in modern humans, but differ from the Neanderthal versions.
                            </p>
                            <p>
                                Researchers found over 200 of these. Many of them appear to contain genes involved with neural development, including <em>DVRK1A</em> (implicated in Down Syndrome), <em>Neuregulin-3</em> (schizophrenia), and <em>CADPS2</em> and <em>AUTS2</em> (autism). The authors also point out <em>RUNX2</em> is part of a selective sweep; mutations in this gene lead to skeletal deformities in the face and shoulders, areas which differ significantly between humans and Neanderthals.
                            </p>
                            <p>
                                Of the largest 20 regions that have swept through the genomes of modern humans, five don't even contain genes. Given that some of the key differences in the other 15 probably lie outside the coding regions of the genes they contain, the data implicates gene regulation as a significant driver of the evolutionary adaptation in humans.
                            </p>
                            <p>
                                Those of you who followed the selective sweep link above saw that it was part one of a two-article description, laying out evidence that one gene involved in brain development swept through humans after we picked up the Neanderthal copy <a href="http://arstechnica.com/science/news/2006/11/5911.ars">through interbreeding</a>. We've now got the Neanderthal sequence of that gene, and it doesn't look like the human version, so that hypothesis just took a major hit.
                            </p>
                            <p>
                                At the same time, the genome sequence does provide evidence that humans and Neanderthals have interbred. This became apparent when the Neanderthal genome was paired against human genomes from different parts of the globe. The Neanderthal DNA consistently matched European and Asian samples better than it did African; the difference was small, but consistent. It suggested that the Neanderthals, which were restricted to Europe and Asia at the time modern humans originated in Africa, had interbred with humans once they began migrating out of Africa.
                            </p>
                            <p>
                                Because African human populations are older, they tend to have more divergent genomes. But the human-Neanderthal split is older still, so the authors figured that any areas of the genome where variation was larger in populations outside of Africa may have entered the human genome through interbreeding. If they did arise through interbreeding, then the non-African segments should match Neanderthals. Researchers found at least 10 regions that fit these predictions.
                            </p>
                            <p>
                                Although they can't rule out the possibility that modern humans had already started diverging from Neanderthals before leaving Africa, the research team favors the idea of interbreeding in the Mid-East as the first modern humans left Africa. This would ensure that both the Asian and European populations picked up some Neanderthal DNA.
                            </p>
                            <h3>
                                Some technical details
                            </h3>
                            <p>
                                Perhaps the most amazing part of the work is that the authors wanted to have a bit more perspective on the geographic differences among modern human populations, so they just did a draft of five more human genomes in order to get it. That's where the technology is at now—if it's helpful to have another genome, you can just go out and get it. They also took advantage of some of the other genomes that have been done recently, including Craig Venter's. That leads to some rather unusual phrasing, like "present-day African segments with the lowest divergence to Neandertals have a divergence to Venter that is 35 percent."
                            </p>
                            <p>
                                The other indication of the capacity of modern sequencing machines comes from the fact that there simply wasn't enough Neanderthal DNA to completely fill a machine's capacity. The team had to attach short sequences that acted as barcodes to identify the Neanderthal DNA, and ran it at the same time as other sequencing work in order to avoid wasting time and money by performing a run at less than full capacity.
                            </p>
                            <p>
                                The Max Planck Institute has also developed a base calling algorithm, <a href="http://bioinf.eva.mpg.de/Ibis/">called Ibis</a>, that uses machine learning to help it deal with the sorts of damage that is typical in ancient DNA samples. These can be pretty substantial, too; if a Cytosine appears at the end of a fragment, 40 percent of the time damage had converted it into a thymidine. That number dropped the further from the end you got, while other types of damage showed different profiles. Accounting for all of this is anything but simple, so it's no surprise that software plays a major role.
                            </p>
                            <p>
                                Contamination is still a problem—most of the sequence that comes out is still bacterial—but the authors did pilot runs, and only worked with samples that produced a mix of sequences that was greater than 1.5 percent hominin. To avoid having most of that be human, they did two checks. The first was to assemble the mitochondrial genomes that came out of the data and ensure that those genomes matched Neanderthal, rather than human, samples. The second was to take advantage of the fact that their samples seemed to be females; any sign of a Y-chromosome was an indication of contamination.
                            </p>
                            <p>
                                There's another paper in this edition of <em>Science</em> that indicates there's another way of dealing with contamination: affinity purification. We know that most Neanderthal sequences are identical to humans, so we can just use DNA chips containing human sequences to purify Neanderthal DNA by affinity. Right now, we can't make a chip big enough to host the entire human genome (much less both strands of the double helix for the whole genome), so it's limited to fishing out specific stretches of the genome.
                            </p>
                            <p>
                                In this particular paper, the authors focused on purifying the specific places in protein-coding DNA that contain differences between humans and chimps. By having the Neanderthal sequence as well, we can determine how many of these are unique to modern humans.
                            </p>
                            <p>
                                <em>Science</em>, 2010. DOI: <a href="http://dx.doi.org/10.1126/science.1188021">10.1126/science.1188021</a> &nbsp;(<a href="http://arstechnica.com/science/news/2010/03/dois-and-their-discontents-1.ars">About DOIs</a>).
                            </p>
                            <p>
                                <a href="http://arstechnica.com/science/news/2010/05/neanderthal-genome-draft-done-indicates-interbreeding-occurred.ars?utm_source=rss&amp;utm_medium=rss&amp;utm_campaign=rss&amp;comments=1#comments-bar">Read the comments on this post</a>
                            </p>
                        </blockquote>
                    </div>
                    <p class="bookmark-source">
                        Source: <a href="http://arstechnica.com/science/news/2010/05/neanderthal-genome-draft-done-indicates-interbreeding-occurred.ars?utm_source=rss&amp;utm_medium=rss&amp;utm_campaign=rss">http://arstechnica.com/science/news/2010/05/neanderthal-genome-draft-done-indicates-interbreeding-occurred.ars?utm_source=rss&amp;utm_medium=rss&amp;utm_campaign=rss</a>
                    </p>
                </div>
            </article>
            <nav id="post-nav"></nav><script id="discus-javascript">
    var disqus_shortname = 'improbable';

                (function() {
                    var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
                    dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
                    document.getElementsByTagName('body')[0].appendChild(dsq);
                })();
            </script><a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>
            <div id="disqus_thread"></div>
        </section>
        <footer id="site-footer" class="nocontent">
            <p>
                This site is purely my personal work and does not reflect the views of my employer.
            </p>
            <p class="license">
                <a class="icon" rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US"><img alt="Creative Commons License" style="border-width:0" src="http://i.creativecommons.org/l/by-sa/3.0/88x31.png"></a> This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US">Creative Commons Attribution-ShareAlike 3.0 Unported License</a>.
            </p>
        </footer><script async="" defer src="/static/js/common.js">
</script><script id="google-analytics">
    var _gaq = _gaq || [];
            _gaq.push(['_setAccount', 'UA-2097834-1']);
            _gaq.push(['_setDomainName', 'improbable.org']);
            _gaq.push(['_trackPageview']);

            (function() {
                var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
                ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
                var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
            })();
        </script>
    </body>
</html>
