<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title>
            Sons of the conquerors: the story of India? | Gene Expression
        </title>
        <meta name="viewport" content="width=device-width,initial-scale=1.0">
        <link rel="stylesheet" href="/static/css/main.min.css" type="text/css" media="all"><!--[if lte IE 8]>
            <link rel="stylesheet" href="/static/css/ie-fixes.css" type="text/css" media="all">
            <script src="/static/js/html5shiv.js"></script>
        <![endif]-->
        <meta http-equiv="last-modified" content="Thu, 28 Oct 2010 09:43:22 GMT">
        <link rel="alternate" type="application/atom+xml" title="All Posts (Atom)" href="/feeds/all.atom">
        <link rel="alternate" type="application/rss+xml" title="All Posts (RSS)" href="/feeds/all.rss">
        <link rel="icon" type="image/jpeg" sizes="287x287" href="/static/img/favicon-287x287.jpg">
        <link rel="icon" type="image/jpeg" sizes="128x128" href="/static/img/favicon-128x128.jpg">
        <script type="application/javascript">
var _prum={id:"5166be01e6e53d1007000001"};var PRUM_EPISODES=PRUM_EPISODES||{};PRUM_EPISODES.q=[];PRUM_EPISODES.mark=function(b,a){PRUM_EPISODES.q.push(["mark",b,a||new Date().getTime()])};PRUM_EPISODES.measure=function(b,a,b){PRUM_EPISODES.q.push(["measure",b,a,b||new Date().getTime()])};PRUM_EPISODES.done=function(a){PRUM_EPISODES.q.push(["done",a])};PRUM_EPISODES.mark("firstbyte");(function(){var b=document.getElementsByTagName("script")[0];var a=document.createElement("script");a.type="text/javascript";a.async=true;a.charset="UTF-8";a.src="//rum-static.pingdom.net/prum.min.js";b.parentNode.insertBefore(a,b)})();
        </script>
    </head>
    <body class="blog post_detail" itemscope itemtype="http://schema.org/BlogPosting">
        <nav id="site-nav">
            <header id="about">
                <h1>
                    <a href="/about/">Chris Adams</a>
                </h1>
                <h2>
                    Programmer, cyclist, photographer
                </h2>
            </header>
            <ul class="links">
                <li>
                    <a href="/">Home</a>
                </li>
                <li>
                    <a href="/about/">About</a>
                </li>
                <li>
                    <a href="/feeds/all.atom" title="Site Feed">Site Feed</a>
                </li>
            </ul>
            <ul class="social-networks">
                <li>
                    <a href="https://github.com/acdha" rel="me">Github</a>
                </li>
                <li>
                    <a href="http://bitbucket.org/acdha" rel="me">Bitbucket</a>
                </li>
                <li>
                    <a href="http://delicious.com/acdha" rel="me">del.icio.us</a>
                </li>
                <li>
                    <a href="http://twitter.com/acdha" rel="me">Twitter</a>
                </li>
                <li>
                    <a href="https://plus.google.com/116562742092842686896?rel=author" rel="me">Google+</a>
                </li>
                <li>
                    <a href="http://www.flickr.com/photos/acdha" rel="me">Flickr</a>
                </li>
                <li>
                    <a href="http://www.linkedin.com/in/acdha" rel="me">LinkedIn</a>
                </li>
                <li>
                    <a href="http://connect.garmin.com/explore?owner=acdha" rel="me">Garmin Connect</a>
                </li>
            </ul>
            <div id="site-search"></div>
        </nav>
        <section id="main">
            <article class="post">
                <header>
                    <meta itemprop="dateCreated" content="2010-10-28T05:43:22-04:00">
                    <meta itemprop="dateModified" content="2010-10-28T05:43:22-04:00"><time class="date" itemprop="datePublished" datetime="2010-10-28T09:43:22+00:00">Oct 28</time>
                    <h2 itemprop="title">
                        Sons of the conquerors: the story of India? | Gene Expression
                    </h2>
                </header>
                <div class="body" itemprop="articleBody">
                    <div class="googlereader description" data-google-id="tag:google.com,2005:reader/item/79b7be2489de38de">
                        <blockquote>
                            <p>
                                <img title="munda2" src="http://blogs.discovermagazine.com/gnxp/files/2010/10/munda2.png" alt="munda2" width="600" height="305">
                            </p>
                            <p>
                                The past ten years has obviously been very active in the area of human genomics, but in the domain of South Asian genetic relationships in a world wide context it has seen veritable revolutions and counter-revolutions. The final outlines are still to be determined. In the mid-1990s the conventional wisdom was that South Asians were a branch of a broader West Eurasian cluster of peoples, albeit more distant from the core Middle Eastern-North-African-European-Caucasian clade. The older physical anthropological literature would have asserted that South Asians were predominantly Caucasoid, but with a <a rel="nofollow" href="http://en.wikipedia.org/wiki/Australoid_race">Australoid</a> element admixed in at varying proportions as a function of geography and caste. To put it more concretely, and I think accurately, a large degree of South Asian physical variety can be defined along the spectrum between <a rel="nofollow" href="http://en.wikipedia.org/wiki/A._R._Rahman">A. R. Rahman</a> and <a rel="nofollow" href="http://en.wikipedia.org/wiki/Nawaz_Sharif">Nawaz Sharif</a>. The regional and caste truisms are only correlations. <a rel="nofollow" href="http://en.wikipedia.org/wiki/Subrahmanyan_Chandrasekhar">Subrahmanyan Chandrasekhar</a> was a Tamil Brahmin, but experienced anti-black racism in the United States. I think that is reasonable in light of his <a rel="nofollow" href="http://images.google.com/images?hl=en&amp;source=imghp&amp;biw=1366&amp;bih=632&amp;q=Subrahmanyan+Chandrasekhar&amp;gbv=2&amp;aq=f&amp;aqi=&amp;aql=&amp;oq=&amp;gs_rfai=">appearance</a>.
                            </p>
                            <p>
                                <span style="float:left;padding:5px"><a rel="nofollow" href="http://www.researchblogging.org"><img style="border:0" src="http://www.researchblogging.org/public/citation_icons/rb2_large_gray.png" alt="ResearchBlogging.org"></a></span>This rough &amp; ready mainstream understanding, supporting by <a rel="nofollow" href="http://www.amazon.com/exec/obidos/ASIN/0691029059/geneexpressio-20">classical genetic markers</a>, was overturned in the early years of the 21st century. One line of thought <strong>argued that South Asians were much more distinctive from the broader Western Eurasian cluster of peoples.</strong> Representative of this body of work is a paper like <a rel="nofollow" href="http://www.ncbi.nlm.nih.gov/pubmed/12536373">The genetic heritage of the earliest settlers persists both in Indian tribal and caste populations</a>. These researchers tended to start with the female lineages, mtDNA, and then supplement that with Y lineages, the paternal descent. A separate line of evidence, generally drawn from <a rel="nofollow" href="http://www.gnxp.com/blog/2009/11/r1a1-and-peopling-of-eurasia.php">Y chromosomal results</a>, <strong>indicated that there were deep connections between the people of India and those of Central Eurasia</strong>, in particular via the <a rel="nofollow" href="http://en.wikipedia.org/wiki/Haplogroup_R1a_(Y-DNA)">R1a</a> haplogroup. Additionally, one aspect of the first set of results which was very surprising was that it actually placed <a rel="nofollow" href="http://blogs.discovermagazine.com/gnxp/2008/12/south-indian-phylogeography/">South Asians closer to</a> East, not West, Eurasians. But by the end of the aughts the uniparental studies had been supplemented by a range of results produced from SNP-chips,<a rel="nofollow" href="http://www.sciencemag.org/cgi/content/abstract/319/5866/1100?maxtoshow=&amp;HITS=10&amp;hits=10&amp;RESULTFORMAT=&amp;fulltext=mYERS+rm&amp;searchid=1&amp;FIRSTINDEX=0&amp;sortspec=date&amp;resourcetype=HWCIT">which looked at hundreds of thousands of genetic variants</a>. These studies seemed to support the older view of South Asians being closer to West Eurasians than East Eurasians. Finally <a rel="nofollow" href="http://www.nature.com/nature/journal/v461/n7263/abs/nature08365.html">last year</a> a paper came out which posited that almost all South Asian populations were actually an ancient stabilized hybrid between two groups, a European-like population, “Ancient North Indians” (ANI), and another group which is no longer present in unadmixed form, “Ancient South Indians” (ASI), of whom the Andaman Islanders are distant relatives. Though there was a slight bias toward ANI as a whole, <a rel="nofollow" href="http://blogs.discovermagazine.com/gnxp/2009/09/visualizing-caste-linguistic-differences-in-india-fst/">the fraction of ASI</a> increased as one went southeast, and down the caste ladder. The distinctive “South Asian” ancestral group in other words then may actually be conceived of as a compound of these two elements; an admixture of the native substrate against a European-like genetic background.
                            </p>
                            <p>
                                Strangely it sounds an awful lot like the older idea of a Caucasoid population with Australoid admixture. We <a rel="nofollow" href="http://blogs.discovermagazine.com/gnxp/2010/08/50000-years-of-dreamtime/">know now</a> that the connection between the tribal peoples of India, and the indigenous groups of South and Southeast Asia as a whole, to those of Australia and Melanesia, is tenuous at best. So the term “Australoid” is not really informative, and may even mislead. And in terms of historical linguistics I don’t think we’ve solved the problem by appealing to an “Aryan invasion.” The high fraction of ANI among South Indian tribal groups who are isolated from even Dravidian caste groups is a clue to the likelihood that the admixture event is very ancient, and probably precedes the arrival of the Aryans to the Indian subcontinent.
                            </p>
                            <p>
                                But there are more than two actors in this game. In <a rel="nofollow" href="http://www.nature.com/nature/journal/v461/n7263/abs/nature08365.html">Reconstructing Indian population history</a> the authors acknowledge that their model is stylized, that reality is more complex. Additionally, they perceive in their data that some tribal groups from northeast India have an element which is outside of the purview of a two-way admixture event. They discarded this set from their broader analysis because this seemed to be a restricted phenomenon to these groups. A new paper in <em>Molecular Biology and Evolution</em> re-injects this third element into the picture. <a rel="nofollow" href="http://mbe.oxfordjournals.org/content/early/2010/10/26/molbev.msq288.short?rss=1">Population Genetic Structure in Indian Austroasiatic speakers: The Role of Landscape Barriers and Sex-specific Admixture</a>:
                            </p>
                            <blockquote>
                                <p>
                                    The geographic origin and time of dispersal of Austroasiatic (AA) speakers, presently settled in South and Southeast Asia, remains disputed. Two rival hypotheses, both assuming a demic component to the language dispersal, have been proposed. The first of these places the origin of Austroasiatic speakers in Southeast Asia with a later dispersal to South Asia during the Neolithic, whereas the second hypothesis advocates pre-Neolithic origins and dispersal of this language family from South Asia. To test the two alternative models this study combines the analysis of uniparentally inherited markers with 610,000 common SNP loci from the nuclear genome. Indian AA speakers have high frequencies of Y chromosome haplogroup O2a; our results show that this haplogroup has significantly higher diversity and coalescent time (17-28 KYA) in Southeast Asia, strongly supporting the first of the two hypotheses. Nevertheless, the results of principal component and “<em>structure</em>-like” analyses on autosomal loci also show that the population history of AA speakers in India is more complex, being characterised by two ancestral components – o<strong>ne represented in the pattern of Y chromosomal and EDAR results, the other by mtDNA diversity and genomic structure. We propose that AA speakers in India today are derived from dispersal from Southeast Asia, followed by extensive sex-specific admixture with local Indian populations.</strong>
                                </p>
                            </blockquote>
                            <p>
                                Some background is necessary here. South Asia is notoriously linguistically diverse, but, that diversity can be bracketed into several broad families. First, the Indo-European languages are represented by Indo-Aryan and Iranian dialects (and Germanic, if you include English). Second, the <a rel="nofollow" href="http://en.wikipedia.org/wiki/Dravidian_languages">Dravidian</a> languages are found across the subcontinent, from Brahui in Pakistan to Malto in Bangladesh. But they’re really the dominant languages in the southern cone of South Asia. That being said it seems likely that historically their distribution extended far into the north, with Brahui in western Pakistan being a relic of that period, as well as the fragmented tribal groups in Central India. There is also evidence down to historic periods of a Dravidian-speaking substrate in Maharashtra. <strong>And purely from a philological perspective it seems clear that many Indo-Aryan languages evolved within a Dravidian linguistic substrate</strong>.
                            </p>
                            <p>
                                Next, in the far north there are languages of Tibetan provenance and affinity. These are explicable in their origins and relationship. But in the northeast third of the Indian subcontinent there are a two groups of <a rel="nofollow" href="http://en.wikipedia.org/wiki/Austro-Asiatic_languages">Austro-Asiatic</a> languages. The prefix “Austro” is indicative of the symbiotic relationship between historical linguistics and physical anthropology in the early 20th century (most famously illustrated in the transplantation of the social-linguistic term Aryan from a South Asian and Iranian context, to a racialized Northern European term). The map at the top of this post shows the distribution of the Austro-Asiatic languages, as well as their subdivisions. There is clearly an eastern and western wing to the group, but most scholars assume that this is an artifact of the historical eruption of the Burman and Thai peoples out of the southern fringes of the Chinese Empire and into mainland Southeast Asia.
                            </p>
                            <p>
                                <img style="float:left;padding:4px;margin:0 7px 2px 0" title="800px-Ramakrishna_Mission_Cherrapunjee_106" src="http://blogs.discovermagazine.com/gnxp/files/2010/10/800px-Ramakrishna_Mission_Cherrapunjee_106.JPG" alt="800px-Ramakrishna_Mission_Cherrapunjee_106" width="200" height="270">Within India the Austro-Asiatic languages fall into two broad categories: the Munda and the Khasi. The Khasi inhabit the massif which separates Bengal and Assam. Their culture and society is at some variance from the norm in India (they are matrilocal, and animist or Christian). A close relationship to the people to the east is clear in both their language and their physical appearance. The Khasi, and other groups such as the Garo, are of the family of peoples and ethnicities which have arrived from the east and north relatively recently, making the transition from the world of Tibet and Burma to India. This is evident in the face of the Khasi child in the image to the left. Once passing out of their lands of origin these populations have assimilated to different degrees to the Indic domain. The <a rel="nofollow" href="http://en.wikipedia.org/wiki/Tripuri_people">Tripuri</a> people for example retain a Tibeto-Burman language, but are adherents of Vaishnav Hinduism (my own family were once subjects of the <a rel="nofollow" href="http://en.wikipedia.org/wiki/Manikya_Dynasty">Manikya dynasty</a>). The <a rel="nofollow" href="http://en.wikipedia.org/wiki/Ahom#Ahom_people_today">Ahom</a> of Assam were totally assimilated by the Indo-Aryan substrate. Like the Bulgars of Bulgaria their only influence was in the ethnonym that they contributed to their subjects. A quick survey of my own genetics, and those of other South Asians of eastern origin on 23andMe, clearly shows the influence of assimilated Tibeto-Burmans. One Bangladeshi Muslim individual clearly carries an East Asian Y chromosomal haplogroup.
                            </p>
                            <p>
                                The Munda are a somewhat different case. In older historical literature on South Asia there is some consideration that the Munda may be the earliest inhabitants of India; predating the Dravidians. Some readers of South Asian origin also point out that in the early Indo-Aryan language there may be more evidence of Munda, than Dravidian, influence. But the eastern connections of the Munda languages seem clear, albeit less explicable than those of the Khasi or the Tibeto-Burman peoples of the far northeast. If the Munda are the indigenous people then it stands to reason that the Mon-Khmer languages derive from South Asia. On the other hand the vast majority of the Austro-Asiatic languages exist in Southeast Asia, and, the Munda themselves have been hypothesized as being the bearers of rice-culture from the east.
                            </p>
                            <p>
                                This is where genetics comes into play. There has already been evidence of an eastern influence in the genes of the Munda from other researchers, so what this paper does is look at that in detail, instead of discarding it as a minor effect which muddles the broader picture. I’ve reformatted figure 3 to show how the groups relate to each other. On the left is a PCA. Most of the variance is west-east, ~6%, while some of it is north-south, ~1%. On the right is a bar plot generated from ADMIXTURE. I’ve edited out many of the populations. Focus on the Austro-Asiatic groups from India.
                            </p>
                            <p>
                                <img title="munda1" src="http://blogs.discovermagazine.com/gnxp/files/2010/10/munda1.png" alt="munda1" width="600" height="486">
                            </p>
                            <p>
                                In the PCA you see the SE-NW axis of ANI-ASI admixture which is the primary aspect of genetic variation within South Asia. Numerically Dravidian and Indo-Aryan groups along this axis are the vast majority of South Asians. But the Munda and other Austro-Asiatic groups are not trivial; there <a rel="nofollow" href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.0020215">are strong suggestions</a> that the eastern Indo-Aryan groups, Oriya, Bengali, and Assamese, are to some extent shaped by influence from the Austro-Asiatic elements. The closer connection of the Khasi to East Asian populations is clear on the PCA. But the fact that the South Indian samples are further along axis-Y than the Munda are indicative of admixture in the Munda population. Looking at the bar plot that’s clear. The dominant dark-green signature of South Indian ancestry is also predominant among the Munda, and found at non-trivial amounts among Iranian, Khasi, and Southeast Asian populations, but the Munda clearly have an eastern component which is not found in South Indians. This is probably the element which perturbs them on the PCA.
                            </p>
                            <p>
                                But this just tells us the relationships in terms of total genome content. It doesn’t necessarily tells us the historical sequence of admixture events or the direction of migration. In fact the evidence of Indian ancestry in Southeast Asia could be suggesting migration from South Asia to the Southeast Asia (there is plenty of cultural evidence of transmission, though the presumption is that the demographic movements were marginal). They note in the paper that one phenomenon which could be obscuring and confusing our understanding is that much of gene flow occurs through isolation-by-distance (IBD). Village-to-village dynamics. In contrast to this you have folk wanderings, which result in a “leapfrog” aspect. The <a rel="nofollow" href="http://blogs.discovermagazine.com/gnxp/2009/09/yes-uyghurs-are-a-new-hybrid-population/">Hazara and Uyghur</a> are both cases of leapfrogging, as their genetic makeup can’t be explained easily by IBD. So here the connections between the Munda and Southeast Asians, and the broader relationship between Southeast Asians and South Asians, could be IBD, or perhaps reflect deep ancient common ancestry. Perhaps the ASI group spanned the region from the Arabian Sea to the South China sea, and were only later overlain by ANI and East Asian populations.
                            </p>
                            <p>
                                To explore these questions the authors tunneled down to a more fine-grained scale, and looked at uniparental lineages as well as a gene at which recent selection seems to have operated upon East Asians in distinction to other groups, <em>EDAR</em>. Though uniparental lineages are only partially informative in terms of ancestry, they are very amenable to dating because of their haploid inheritance patterns. And the relationships between the branches of the termini can give us historical information.
                            </p>
                            <p>
                                The following figure shows the relationship and distribution of a particular Y chromosomal haplogroup which the Munda carry, and other South Asians tend not to, which connects them to the east:
                            </p>
                            <p>
                                <img title="munda3" src="http://blogs.discovermagazine.com/gnxp/files/2010/10/munda3.png" alt="munda3" width="600" height="396">
                            </p>
                            <p>
                                The haplogroup is O2a (M95). The results from the Y chromosomal data are not clear, though they do seem to reject the model whereby Southeast Asian O2a lineages derive from Indian ones. But it does not seem as if you have a scenario where one founder lineage entered into South Asia from Southeast Asia, there are too many disparate branches of O2a found among Indians. Additionally, the coalescence time (back to last common ancestor) is deeper in Southeast Asia, but still deep in South Asia among the Munda. From this it seems that the origin of Austro-Asiatic languages in South Asia can be rejected, but the details of the emergence of Austro-Asiatic in South Asia can not be clearly perceived as of yet. From what I can gather the authors themselves do not necessarily believe that their results in this domain are robust (insensitive to varying the model’s assumptions even marginally).
                            </p>
                            <p>
                                An interesting point though is that the mtDNA, the female lineage, does not seem to diverge from other South Asians much at all. I find it intriguing that this is the same pattern we see along the major NW-SE axis of variation. It seems that mtDNA lineages unite South Asians, while the Y lineages separate them (by caste and region). The generality has many exceptions, but it points to a peculiar sex mediated admixture process from <em>both</em> the northwest and northeast. Men on the move have reshaped the genetics and culture of South Asia, but the mtDNA lineages still point to an ancient Eurasian group with distant but stronger affinities to the east than the west. The mtDNA are likely the purest distillation of ASI.
                            </p>
                            <p>
                                Finally, they look at frequencies of variants of <em>EDAR</em> among the South Asian groups. <em>EDAR</em> is in some ways diagnostic of East Asian ancestry; it seems that a variant which produces thick straight hair emerged relatively recently among East Asians. Here’s the result from the HGDP browser:
                            </p>
                            <p>
                                <img title="edar1" src="http://blogs.discovermagazine.com/gnxp/files/2010/10/edar1.png" alt="edar1" width="600" height="432">
                            </p>
                            <p>
                                <img style="float:right;padding:4px;margin:0 0 2px 7px" title="edar2" src="http://blogs.discovermagazine.com/gnxp/files/2010/10/edar2-300x206.png" alt="edar2" width="300" height="206">The G allele exhibits co-dominance, so the GA phenotype has intermediate hair-thickness between AA and GG. Haplotype structure based tests of natural selection have indicated that the derived G allele is recent. The map to the right shows the frequency of the derived G variant by population group. The bubble size is proportional to frequency, while the colors represent language groups. Again the Khasi and Tibeto-Burman groups are as you’d expect, they exhibit a relatively high frequency of the derived variant. The Hazara are a group which only came into being within the last 1,000 years through an admixture event. The Tharu seem to have their origins in Nepal’s transitional zone, and all <a rel="nofollow" href="http://blogs.discovermagazine.com/gnxp/2010/07/one-principal-component-to-rule-them-all/">the Nepali populations</a> have significant admixture with Tibetan groups even if they themselves are not Tibetan in language and culture. The interesting result are the Munda. The Dravidian groups lack the derived <em>EDAR</em> variant, as do Indo-European groups without a plausible East Asian source of admixture. But within the Munda the derived variant is found in proportions ~5%. This is far lower than the 60% among the Tibeto-Burmans of the northeast, or the 40% among the Khasi, but it is significant. And this result allows the authors to reject the IBD model of connection for Austro-Asiatic groups, because the Munda harbor the variant which other South Asian groups in their environs do not. Gene flow predicated on linguistic affiliation at such a remove seems implausible, so the most parsimonious explanation is that the Munda languages arrived in India from Southeast Asia as part of a leapfrog folk wandering.
                            </p>
                            <p>
                                But why the low frequency of the derived variant? Obviously the Munda have admixed with the local substrate, so dilution would be one explanation. Another could be that when the Munda left East Asia the frequency was lower. Additionally, whatever selective forces were driving the frequency up may have abated in South Asia, and it could be that there was selection against the derived variant! Whatever the truth of it the existence of the derived <em>EDAR</em> variant among the Munda would be like finding the European <em>LCT</em> variant among an East Asian population: clear evidence of long distance gene flow and population movement.
                            </p>
                            <p>
                                So where does this lead us? First, let me observe that some of the authors on this paper are the same ones who argued for a predominantly indigenous origin for South Asians in the early 2000s based on mtDNA variation. In this paper they seem to be leaning against an indigenous origin for the Munda, or at least refuting the conjecture that the Munda are <em>ur</em>-Indians par excellence. I didn’t go into the details of the coalescence times because they’re rather a mess, but <em>EDAR</em> is probably a “tipping point” in arguing for a relatively recent exogenous origin for the Munda. The strong sex asymmetry in genetic variation is also suggestive, we have plenty of evidence of historical examples of genetic leapfrogs occurring through men-on-the-move. The asymmetry also seems to exist among the Khasi and other Tibeto-Burmans in India’s northeast (figure 2 of the paper).
                            </p>
                            <p>
                                The arguments about the history, culture, and genetics of South Asia have traditionally been disputed along the Aryan-Dravidian axis. I’m not interested in rehashing that aspect, but these data point us to another reality: <strong>on India’s northeast frontier there’s another component</strong>. As an ethnic Bengali myself I’ve always been somewhat aware of this. Some of my relatives and family acquaintances look much more like <a rel="nofollow" href="http://images.google.com/images?hl=en&amp;safe=off&amp;biw=1366&amp;bih=632&amp;gbv=2&amp;tbs=isch:1&amp;sa=1&amp;q=garo+tribe&amp;aq=f&amp;aqi=g1&amp;aql=&amp;oq=&amp;gs_rfai=">Garos</a> than other South Asians. This component is even more evident on the face of Assamese and Nepali, whose languages are Indo-Aryan and religion is Hinduism, but whose appearance bespeaks a more variegated background. On some level South Asians from these regions are aware of their peculiarity, even if it isn’t spoken of much. I have read that in the wake of the victory of Japan over Russia in the early 20th century Bengali intellectuals expressed in public their pride at their Asiatic ancestry. With the rise of China in the 21st century I suspect more South Asians from Nepal, Bengal, and Assam, will rediscover that aspect of their background which links them to the east, and not the west. The genetics is just telling us what we already knew.
                            </p>
                            <p>
                                <strong>Citation:</strong> <span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.jtitle=Mol+Biol+Evol&amp;rft_id=info%3A%2F10.1093%2Fmolbev%2Fmsq288&amp;rfr_id=info%3Asid%2Fresearchblogging.org&amp;rft.atitle=Population+Genetic+Structure+in+Indian+Austroasiatic+speakers%3A+The+Role+of+Landscape+Barriers+and+Sex-specific+Admixture&amp;rft.issn=&amp;rft.date=2010&amp;rft.volume=&amp;rft.issue=&amp;rft.spage=&amp;rft.epage=&amp;rft.artnum=http%3A%2F%2Fmbe.oxfordjournals.org%2Fcontent%2Fearly%2F2010%2F10%2F26%2Fmolbev.msq288.short%3Frss%3D1&amp;rft.au=Gyaneshwer+Chaubey&amp;rft.au=Mait+Metspalu&amp;rft.au=Ying+Choi&amp;rft.au=Reedik+M%C3%A4gi&amp;rft.au=Irene+Gallego+Romero&amp;rft.au=Pedro+Soares&amp;rft.au=Mannis+van+Oven&amp;rft.au=Doron+M.+Behar&amp;rft.au=Siiri+Rootsi&amp;rft.au=Georgi+Hudjashov&amp;rft.au=Chandana+Basu+Mallick&amp;rft.au=Monika+Karmin&amp;rft.au=Mari+Nelis&amp;rft.au=J%C3%BCri+Parik&amp;rft.au=Alla+Goverdhana+Reddy&amp;rft.au=Ene+Metspalu&amp;rft.au=George+van+Driem&amp;rft.au=Yali+Xue&amp;rft.au=Chris+Tyler-Smith&amp;rft.au=Kumarasamy+Thangaraj&amp;rft.au=Lalji+Singh&amp;rft.au=Maido+Remm&amp;rft.au=Martin+B.+Richards&amp;rft.au=Marta+Mirazon+Lahr&amp;rft.au=Manfred+Kayser&amp;rft.au=Richard+Villems&amp;rft.au=Toomas+Kivisild&amp;rfe_dat=bpr3.included=1;bpr3.tags=Anthropology%2CBiology">Gyaneshwer Chaubey, Mait Metspalu, Ying Choi, Reedik Mägi, Irene Gallego Romero, Pedro Soares, Mannis van Oven, Doron M. Behar, Siiri Rootsi, Georgi Hudjashov, Chandana Basu Mallick, Monika Karmin, Mari Nelis, Jüri Parik, Alla Goverdhana Reddy, Ene Metspalu, George van Driem, Yali Xue, Chris Tyler-Smith, Kumarasamy Thangaraj, Lalji Singh, Maido Remm, Martin B. Richards, Marta Mirazon Lahr, Manfred Kayser, Richard Villems, &amp; Toomas Kivisild (2010). Population Genetic Structure in Indian Austroasiatic speakers: The Role of Landscape Barriers and Sex-specific Admixture <span style="font-style:italic">Mol Biol Evol</span> : <a rel="nofollow" href="http://dx.doi.org/10.1093/molbev/msq288">10.1093/molbev/msq288</a></span>
                            </p>
                            <p>
                                Link acknowledgement: <a rel="nofollow" href="http://dienekes.blogspot.com/">Dienekes Pontikos</a>.
                            </p>
                            <p>
                                <b>Addendum:</b> This is more a speculative comment, so I will tack this on to the body of the main post. Here’s my current very tentative model for how South Asians came to be. At some point after the last Ice Age 10,000 years ago the ANI arrived, and hybridized with the ASI, who are descendants of the older original Out of Africa wave to South Asia. After this, but before the Aryans, the Munda arrived from the northeast, and pushed into lands inhabited by ANI-ASI groups. 4,000-3,000 years ago the Indo-Aryans arrive, and impose themselves as an elite on the ANI-ASI hybrid population, before being assimilated biologically and imparting their language to the Indian majority. I don’t know where Dravidian came from, but perhaps it was the language of the ANI (its existence in fragments all across the swath of the northern Indian subcontinent is suggestive, as well as possible connections to ancient Elamite, the language of Bronze Age southwest Iran). Eventually the Aryanized ANI-ASI marginalized the Munda in northeast India and drove them to the highlands. Finally, the Tibeto-Burmans arrived in the historical period.
                            </p>
                            <p>
                                <i>Image Credit: Wikimedia Commons</i>
                            </p><img src="http://feeds.feedburner.com/~r/DiscoverBlogs/~4/w3oWmsZ0I68" height="1" width="1">
                        </blockquote>
                    </div>
                    <p class="bookmark-source">
                        Source: <a href="http://blogs.discovermagazine.com/gnxp/2010/10/sons-of-the-conquerers-the-story-of-india/?utm_source=feedburner&amp;utm_medium=feed&amp;utm_campaign=Feed%3A+DiscoverBlogs+%28Discover+Blogs%29">http://blogs.discovermagazine.com/gnxp/2010/10/sons-of-the-conquerers-the-story-of-india/?utm_source=feedburner&amp;utm_medium=feed&amp;utm_campaign=Feed%3A+DiscoverBlogs+%28Discover+Blogs%29</a>
                    </p>
                </div>
            </article>
            <nav id="post-nav"></nav><script id="discus-javascript">
    var disqus_shortname = 'improbable';

                (function() {
                    var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
                    dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
                    document.getElementsByTagName('body')[0].appendChild(dsq);
                })();
            </script><a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>
            <div id="disqus_thread"></div>
        </section>
        <footer id="site-footer" class="nocontent">
            <p>
                This site is purely my personal work and does not reflect the views of my employer.
            </p>
            <p class="license">
                <a class="icon" rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US"><img alt="Creative Commons License" style="border-width:0" src="http://i.creativecommons.org/l/by-sa/3.0/88x31.png"></a> This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US">Creative Commons Attribution-ShareAlike 3.0 Unported License</a>.
            </p>
        </footer><script async="" defer src="/static/js/common.js">
</script><script id="google-analytics">
    var _gaq = _gaq || [];
            _gaq.push(['_setAccount', 'UA-2097834-1']);
            _gaq.push(['_setDomainName', 'improbable.org']);
            _gaq.push(['_trackPageview']);

            (function() {
                var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
                ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
                var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
            })();
        </script>
    </body>
</html>
