<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title>
            Genetic watersheds on the Great Himalayas | Gene Expression
        </title>
        <meta name="viewport" content="width=device-width,initial-scale=1.0">
        <link rel="stylesheet" href="/static/css/main.min.css" type="text/css" media="all"><!--[if lte IE 8]>
            <link rel="stylesheet" href="/static/css/ie-fixes.css" type="text/css" media="all">
            <script src="/static/js/html5shiv.js"></script>
        <![endif]-->
        <meta http-equiv="last-modified" content="Wed, 20 Oct 2010 09:29:45 GMT">
        <link rel="alternate" type="application/atom+xml" title="All Posts (Atom)" href="/feeds/all.atom">
        <link rel="alternate" type="application/rss+xml" title="All Posts (RSS)" href="/feeds/all.rss">
        <link rel="icon" type="image/jpeg" sizes="287x287" href="/static/img/favicon-287x287.jpg">
        <link rel="icon" type="image/jpeg" sizes="128x128" href="/static/img/favicon-128x128.jpg">
        <script type="application/javascript">
var _prum={id:"5166be01e6e53d1007000001"};var PRUM_EPISODES=PRUM_EPISODES||{};PRUM_EPISODES.q=[];PRUM_EPISODES.mark=function(b,a){PRUM_EPISODES.q.push(["mark",b,a||new Date().getTime()])};PRUM_EPISODES.measure=function(b,a,b){PRUM_EPISODES.q.push(["measure",b,a,b||new Date().getTime()])};PRUM_EPISODES.done=function(a){PRUM_EPISODES.q.push(["done",a])};PRUM_EPISODES.mark("firstbyte");(function(){var b=document.getElementsByTagName("script")[0];var a=document.createElement("script");a.type="text/javascript";a.async=true;a.charset="UTF-8";a.src="//rum-static.pingdom.net/prum.min.js";b.parentNode.insertBefore(a,b)})();
        </script>
    </head>
    <body class="blog post_detail" itemscope itemtype="http://schema.org/BlogPosting">
        <nav id="site-nav">
            <header id="about">
                <h1>
                    <a href="/about/">Chris Adams</a>
                </h1>
                <h2>
                    Programmer, cyclist, photographer
                </h2>
            </header>
            <ul class="links">
                <li>
                    <a href="/">Home</a>
                </li>
                <li>
                    <a href="/about/">About</a>
                </li>
                <li>
                    <a href="/feeds/all.atom" title="Site Feed">Site Feed</a>
                </li>
            </ul>
            <ul class="social-networks">
                <li>
                    <a href="https://github.com/acdha" rel="me">Github</a>
                </li>
                <li>
                    <a href="http://bitbucket.org/acdha" rel="me">Bitbucket</a>
                </li>
                <li>
                    <a href="http://delicious.com/acdha" rel="me">del.icio.us</a>
                </li>
                <li>
                    <a href="http://twitter.com/acdha" rel="me">Twitter</a>
                </li>
                <li>
                    <a href="https://plus.google.com/116562742092842686896?rel=author" rel="me">Google+</a>
                </li>
                <li>
                    <a href="http://www.flickr.com/photos/acdha" rel="me">Flickr</a>
                </li>
                <li>
                    <a href="http://www.linkedin.com/in/acdha" rel="me">LinkedIn</a>
                </li>
                <li>
                    <a href="http://connect.garmin.com/explore?owner=acdha" rel="me">Garmin Connect</a>
                </li>
            </ul>
            <div id="site-search"></div>
        </nav>
        <section id="main">
            <article class="post">
                <header>
                    <meta itemprop="dateCreated" content="2010-10-20T05:29:45-04:00">
                    <meta itemprop="dateModified" content="2010-10-20T05:29:45-04:00"><time class="date" itemprop="datePublished" datetime="2010-10-20T09:29:45+00:00">Oct 20</time>
                    <h2 itemprop="title">
                        Genetic watersheds on the Great Himalayas | Gene Expression
                    </h2>
                </header>
                <div class="body" itemprop="articleBody">
                    <div class="googlereader description" data-google-id="tag:google.com,2005:reader/item/f1da0c90dacee7bd">
                        <blockquote>
                            <p>
                                <img title="Indian_subcontinent" src="http://blogs.discovermagazine.com/gnxp/files/2010/10/Indian_subcontinent.png" alt="Indian_subcontinent" width="600" height="222">
                            </p>
                            <p>
                                One of the great geological landmarks on earth are the Himalayas. Not only are the Himalayas of importance in the domain of physical geography, but they are important in human geography as well. Just as South Asians and non-South Asians agree that the valley of the Indus and its tributaries bound the west of the Indian cultural world, so the Himalayas <a rel="nofollow" href="http://en.wikipedia.org/wiki/Aryavarta">bound it on the north</a>. Unlike many pre-modern constructions, such as the <a rel="nofollow" href="http://en.wikipedia.org/wiki/Europe#Definition">eastern boundary</a> of Europe, the northern limit of South Asia is relatively clear and distinct. It is stark on a relief map; the flat Gangetic plain gives way to mountains. And it is stark a cultural map, the languages of northern India give way to those of the world of Tibet. The religion of northern India gives way to the Buddhism of Tibet. In terms of human geography I believe that one can argue that the Himalayan fringe around South Asia exhibits the greatest change of ancestrally informative gene frequencies over the smallest distance when you exclude those regions separated by water barriers. Unlike the Sahara the transect from the northern India to Chinese Tibet at any given point along the border is <em>permanently</em> inhabited, <a rel="nofollow" href="http://en.wikipedia.org/wiki/Ladakh">albeit sparsely</a> at the heights.
                            </p>
                            <p>
                                <span style="float:left;padding:5px"><a rel="nofollow" href="http://www.researchblogging.org"><img style="border:0" src="http://www.researchblogging.org/public/citation_icons/rb2_large_gray.png" alt="ResearchBlogging.org"></a></span>And yet despite the geographical barriers people and ideas did move across the Himalaya. The cultural influences upon Tibet from India are obvious. The <a rel="nofollow" href="http://en.wikipedia.org/wiki/Tibetan_script#History">script</a> of Tibet is derived from India, while its form of Buddhism is the <a rel="nofollow" href="http://en.wikipedia.org/wiki/Tibetan_Buddhism#Early_history">direct descendant</a> of the last efflorescence&nbsp;of that religion in northern India. But while culture moved north, I do not see much evidence genetically that South Asians have been significant as an influence. This is somewhat shocking when you realize these two facts: the population of the <a rel="nofollow" href="http://en.wikipedia.org/wiki/Tibet_Autonomous_Region#Demographics">Tibetan Autonomous Region</a> is on the order of 5-6 million, while that of northern South Asia around <a rel="nofollow" href="http://en.wikipedia.org/wiki/Indo-Gangetic_Plain">~1 billion</a> (including Pakistan and Bangladesh). A 200-fold difference. And yet there is evidence of admixture between the two groups exactly where you’d expect: in Nepal. Below is a figure from a <a rel="nofollow" href="http://www.ncbi.nlm.nih.gov/pubmed/20643205">recent paper</a> which shows how South and East Asian populations relate to each other. I’ve highlighted the Nepali groups, which span the two larger classes:
                            </p>
                            <p>
                                <br>
                                <img title="tibetsou1" src="http://blogs.discovermagazine.com/gnxp/files/2010/10/tibetsou1.png" alt="tibetsou1" width="600" height="417">
                            </p>
                            <p>
                                <img style="float:left;padding:4px;margin:0 7px 2px 0" title="tibetsouth2" src="http://blogs.discovermagazine.com/gnxp/files/2010/10/tibetsouth2.png" alt="tibetsouth2" width="125" height="285">From the above figure it’s clear that there is considerable admixture among the Indo-European populations of Nepal with a Tibetan element. The <a rel="nofollow" href="http://en.wikipedia.org/wiki/Magar_language">Magar</a> are a tribe which is representative of Tibet, with little South Asian genetic input presumably. The <a rel="nofollow" href="http://en.wikipedia.org/wiki/Newar">Newar</a> are the Nepalese hybrids <em>par excellence</em>. To a great extent they can be viewed as the indigenous peoples of the Kathmandu region at the heart of modern Nepal. Their language is of Tibetan affinity, and yet it is heavily overlain with an Indo-Aryan aspect, and seems to have within it an ancient Austro-Asiatic substrate. Though predominantly Hindu today, the Newar have a substantial Buddhist minority whose roots may go back to the original Mahayana traditions which were once prominent in northern India. The Brahmin and Chetri groups are upper caste communities who claim provenance from the north Indian plain. Some of these upper caste groups in Nepal are of recent vintage, having fled the Islamic conquests of the Gangetic plain within the last 1,000 years. <strong>And yet even they have obvious Tibetan admixture.</strong> This should make one cautious about the excessive claims to genetic purity which South Asian caste groups make.
                            </p>
                            <p>
                                But admixture of a Tibetan or East Asian component in South Asia is not limited to Nepal. I have reedited a figure from a <a rel="nofollow" href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.0020215">2006 paper on Indian Americans</a> which shows the inferred components of ancestry of various language-groups. It is clear that the northeastern groups, Bengalis, Assamese, and Oriya, have an affinity to East Asians. This is not just ancient east Eurasian ancestry, the “Ancestral South Indians” hypothesized in <a rel="nofollow" href="http://www.nature.com/nature/journal/v461/n7263/abs/nature08365.html">Reich et al.</a>. The South Indian groups (which I have excised from the figure) do not exhibit the same level of elevation of the ancestral quantum dominant among the Han Chinese in the bar plot. In fact the <a rel="nofollow" href="http://www.nature.com/nature/journal/v461/n7263/abs/nature08365.html">Reich et al.</a> paper also reported evidence of an eastern ancestral element in some of the Munda speaking groups of northeast India. This stands to reason as the Munda are a South Asian branch of the <a rel="nofollow" href="http://en.wikipedia.org/wiki/Austro-Asiatic_languages">Austro-Asiatic family</a> of Southeast Asia. But much of it may also be more recent, as groups such as the Ahom of Assam and the Chakma of Bangladesh seem to have arrived from Burma of late.
                            </p>
                            <p>
                                So we see that genes do flow around the margins of South Asia, and into it. And yet Tibet seems oddly insulated. Why? Because of adaptation. Like water, it seems in this case genes tend to flow downhill, not up, and the reason is likely the fitness differentials between lowland and highland populations along the slope of the Great Himalayas. A new paper in <em>PNAS</em> explores the issue by examining genetic variation among Indians, Tibetans, and worldwide populations, in relation to hypoxia implicated loci. <a rel="nofollow" href="http://www.pnas.org/content/early/2010/10/14/1006108107.abstract"><em>EGLN1</em> involvement in high-altitude adaptation revealed through genetic analysis of extreme constitution types defined in Ayurveda</a>:
                            </p>
                            <blockquote>
                                <p>
                                    It is being realized that identification of subgroups within normal controls corresponding to contrasting disease susceptibility is likely to lead to more effective predictive marker discovery. We have previously used the Ayurvedic concept of&nbsp;<em>Prakriti</em>, which relates to phenotypic differences in normal individuals, including response to external environment as well as susceptibility to diseases, to explore molecular differences between three contrasting&nbsp;<em>Prakriti</em> types:&nbsp;<em>Vata, Pitta, and Kapha</em><em>. EGLN1</em> was one among 251 differentially expressed genes between the&nbsp;<em>Prakriti</em> types. In the present study, we report a link between high-altitude adaptation and common variations rs479200 (C/T) and rs480902 (T/C) in the&nbsp;<em>EGLN1</em> gene. Furthermore, the TT genotype of rs479200, which was more frequent in&nbsp;<em>Kapha</em> types and correlated with higher expression of&nbsp;<em>EGLN1</em>, was associated with patients suffering from high-altitude pulmonary edema, whereas it was present at a significantly lower frequency in&nbsp;<em>Pitta</em> and nearly absent in natives of high altitude. Analysis of Human Genome Diversity Panel-Centre d’Etude du Polymorphisme Humain (HGDP-CEPH) and Indian Genome Variation Consortium panels showed that disparate genetic lineages at high altitudes share the same ancestral allele (T) of rs480902 that is overrepresented in&nbsp;<em>Pitta</em> and positively correlated with altitude globally (<em>P</em>&lt; 0.001), including in India. <strong>Thus,</strong> <em><strong>EGLN1</strong></em> <strong>polymorphisms are associated with high-altitude adaptation, and a genotype rare in highlanders but overrepresented in a subgroup of normal lowlanders discernable by Ayurveda may confer increased risk for high-altitude pulmonary edema.</strong>
                                </p>
                            </blockquote>
                            <p>
                                The paper itself is a follow up to a previous work attempting to see if there was a sense to the classification of constitutions found within <a rel="nofollow" href="http://en.wikipedia.org/wiki/Ayurveda">Ayurvedic medicine</a>. Like Chinese medicine this is a non-Western tradition which has different philosophical roots and axioms (Galenic medicine might be analogous). But in theory all medical traditions emerged to battle illness, so their target was unitary, the ailments which plague the human body. Therefore one might suppose that in fact there would be some sense in any long-standing medical tradition which has any empirical grounding, because human biology is <em>relatively</em> invariant.&nbsp;It is the <em>relative</em> clause which is of interest for the purposes of this paper, <strong>because the authors show how the classifications of Ayurvedic medicine seem to comport with the recent genetic evidence of high altitude adaptation!</strong> Specifically they found that particular Ayurvedic classes of individuals who seem to have negative reactions to high altitude exposure in the form of hypoxia tend to be carriers of particular <em>EGLN1</em> genotypes.
                            </p>
                            <p>
                                I will at this point observe that since I don’t know much about Ayurveda I won’t address or cover that issue in detail. The paper is Open Access so you can read it yourself. So let’s move to the genetics. <em>EGLN1</em> should be familiar to you by now. It’s cropped up repeatedly over the past year in <a rel="nofollow" href="http://blogs.discovermagazine.com/gnxp/2010/09/the-silver-age-of-altitude-adaptation/">studies of</a> <a rel="nofollow" href="http://blogs.discovermagazine.com/gnxp/2010/07/why-tibetans-breath-so-easy-up-high/">high altitude</a> <a rel="nofollow" href="http://blogs.discovermagazine.com/gnxp/2010/05/breathing-like-buddha-altitude-tibet/">adaptation</a>. It is a locus which seems to be a target of selection in <em>both</em> the peoples of the Andes and Tibet. Additionally, it has a peculiar aspect where the ancestral variant, the one found most frequently within Africa, seems to be the target of selection for altitude adaptation outside of Africa.
                            </p>
                            <p>
                                The slideshow below is an overview of the primary figures within this paper.
                            </p>
                            <hr>
                            <div>
                                <img title="Frequency of T allele rs480902 in people of different Ayurvedic constitutions" alt="&lt;div&gt;&lt;strong&gt;This is a South Asian sample from the original study group, and the constitution types are Kapha (K), Pitta (P), Vata (V). IE = Indo-European pooled sample, while VPK is the different constitutions pooled. Those who suffer from high altitude pulmonary edema (HAPE) are more likely to be TT genotype.&lt;/strong&gt;&lt;/div&gt;" src="http://blogs.discovermagazine.com/gallery/albums/razib-s-next-gallery/engl6.png"><img title="Frequency of T allele rs480902 - South Asia" alt="&lt;strong&gt;On this marker in &lt;em&gt;EGLN1&lt;/em&gt; the T allele has been shown to be correlated with high altitude adaptation.&lt;/strong&gt;" src="http://blogs.discovermagazine.com/gallery/albums/razib-s-next-gallery/egln1.png"><img title="Map of T allele of rs480902 - South Asia" alt="&lt;strong&gt;On this map the lighter shading indicates a higher frequency of the T allele. Remember that most of the area is interpolated and likely sensitive to representativeness of sample and population coverage.&lt;/strong&gt;" src="http://blogs.discovermagazine.com/gallery/albums/razib-s-next-gallery/egln2.png"><img title="Frequency of T allele rs480902 - World" alt="&lt;strong&gt;Observe the difference between Africa and the rest of the world. The T allele is ancestral, and was presumably at high frequency within Africa. It dropped in frequency outside of Africa, but seems elevated among high altitude groups.&lt;/strong&gt;" src="http://blogs.discovermagazine.com/gallery/albums/razib-s-next-gallery/engl3.png"><img title="T allele rs480902 and altitude correlation - World" alt="&lt;strong&gt;The a-axis is altitude in meters, the y-axis proportion of the T allele. Note that the samples come from both the Indian Genome Variation Consortium and the Human Genome Diversity Project. Additionally observe that the R-squared indicates the proportion of variation in y which can be explained by variation in x.&lt;/strong&gt;" src="http://blogs.discovermagazine.com/gallery/albums/razib-s-next-gallery/egln4.png"><img title="Map of T allele from HGDP browser" alt="&lt;strong&gt;This map shows the frequencies only in the HGDP populations. So the primary Indian and high altitude groups are not shown here.&lt;/strong&gt;" src="http://blogs.discovermagazine.com/gallery/albums/razib-s-next-gallery/egln5.png">
                                <hr>
                            </div>
                            <hr>
                            <p>
                                What do we take away from this? <strong>Well, one aspect which I think is important to emphasize is that genetic background matters, and there’s much we don’t know.</strong> In the conclusion the authors note that the altitude adaptation papers which I alluded to above were not published when the manuscript was being written, so they were not privy to the rather repeated robust evidence that <em>EGLN1</em> has been the target of natural selection, and that variation on the locus is correlated with variation in adaptation to higher altitudes. The widespread coverage of populations in this paper seems to almost obscure as much as highlight. What has African variation to do with this after all? Additionally one must always remember that one given marker on a gene which shows a <em>correlation</em> does not entail functional causation. We saw this with the <a rel="nofollow" href="http://en.wikipedia.org/wiki/OCA2">markers</a> which seemed to predict the odds of an individual of European ancestry having blue eyes; it turned out that the markers themselves were simply strongly associated with another SNP which was probably the real functional root behind the difference in phenotype.
                            </p>
                            <p>
                                Due to the replication of <em>EGLN1</em> in both Andeans and Tibetans I am moderately confident that variation on this gene does have something to due with high altitude adaptation. What I am curious about is the fact that the ancestral alleles in many cases seem to be driven up on frequency. Is there an interaction between the genetic background of non-Africans and the SNPs in question which make it beneficial toward altitude adaptation? Was there an initial relaxation of function as human populations moved out of Africa, which was slammed back on at high altitudes? There does seem a correlation within South Asian populations between <a rel="nofollow" href="http://en.wikipedia.org/wiki/Hypoxia_(medical)">hypoxia and high altitudes</a> and particular variants on <em>EGLN1</em>. Focusing just on this region we can draw some reasonable inferences, but taking a bigger picture view and encompassing the whole world we’re confronted with a rather more confused, and perhaps more interesting, picture.
                            </p>
                            <p>
                                Back to the specific issue of the lack of South Asian imprint on the genes of Tibetan peoples, I think one can chalk this up to the fact that humans are animals, and so we’re constrained by geography and biology. Tibetans can operate efficiently at lower altitudes, and so have mixed with South Asians in these regions. In contrast, South Asians can not operate at higher altitudes, and so the impact on Tibetans was purely cultural, and not genetic. More broadly this may also point to long term geopolitical implications: the Han Chinese demographic domination of Tibet is always going to be a matter of water flowing uphill. Unless of course we flesh out the genetic architecture of these traits well enough that the Chinese government knows exactly which individuals among the 1.2 billion Han population would be most biologically prepared to reside in the Tibetan Autonomous Region, and so can proactively recruit them to settle in Lhasa and other strategic locations.
                            </p>
                            <p>
                                <strong>Citation:</strong> <span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.jtitle=PNAS&amp;rft_id=info%3A%2F10.1073%2Fpnas.1006108107&amp;rfr_id=info%3Asid%2Fresearchblogging.org&amp;rft.atitle=EGLN1+involvement+in+high-altitude+adaptation%0D%0Arevealed+through+genetic+analysis+of+extreme%0D%0Aconstitution+types+defined+in+Ayurveda%0D%0A&amp;rft.issn=&amp;rft.date=2010&amp;rft.volume=&amp;rft.issue=&amp;rft.spage=&amp;rft.epage=&amp;rft.artnum=http%3A%2F%2Fwww.pnas.org%2Fcontent%2Fearly%2F2010%2F10%2F14%2F1006108107.abstract&amp;rft.au=Shilpi+Aggarwal&amp;rft.au=Sapna+Negi&amp;rft.au=Pankaj+Jha&amp;rft.au=Prashant+K.+Singh&amp;rft.au=Tsering+Stobdan&amp;rft.au=M.+A.+Qadar+Pasha&amp;rft.au=Saurabh+Ghosh&amp;rft.au=Anurag+Agrawal&amp;rft.au=Indian+Genome+Variation+Consortium&amp;rft.au=Bhavana+Prasher&amp;rft.au=Mitali+Mukerji&amp;rfe_dat=bpr3.included=1;bpr3.tags=Biology">Shilpi Aggarwal, Sapna Negi, Pankaj Jha, Prashant K. Singh, Tsering Stobdan, M. A. Qadar Pasha, Saurabh Ghosh, Anurag Agrawal, Indian Genome Variation Consortium, Bhavana Prasher, &amp; Mitali Mukerji (2010). EGLN1 involvement in high-altitude adaptation revealed through genetic analysis of extreme constitution types defined in Ayurveda <span style="font-style:italic">PNAS</span> : <a rel="nofollow" href="http://dx.doi.org/10.1073/pnas.1006108107">10.1073/pnas.1006108107</a></span>
                            </p>
                            <p>
                                <i>Image Credit: Wikimedia Commons</i>
                            </p><img src="http://feeds.feedburner.com/~r/DiscoverBlogs/~4/DRgzu-5Yq5E" height="1" width="1">
                        </blockquote>
                    </div>
                    <p class="bookmark-source">
                        Source: <a href="http://blogs.discovermagazine.com/gnxp/2010/10/genetic-watersheds-on-the-great-himalaya/?utm_source=feedburner&amp;utm_medium=feed&amp;utm_campaign=Feed%3A+DiscoverBlogs+%28Discover+Blogs%29">http://blogs.discovermagazine.com/gnxp/2010/10/genetic-watersheds-on-the-great-himalaya/?utm_source=feedburner&amp;utm_medium=feed&amp;utm_campaign=Feed%3A+DiscoverBlogs+%28Discover+Blogs%29</a>
                    </p>
                </div>
            </article>
            <nav id="post-nav"></nav><script id="discus-javascript">
    var disqus_shortname = 'improbable';

                (function() {
                    var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
                    dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
                    document.getElementsByTagName('body')[0].appendChild(dsq);
                })();
            </script><a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>
            <div id="disqus_thread"></div>
        </section>
        <footer id="site-footer">
            <p>
                This site is purely my personal work and does not reflect the views of my employer.
            </p>
            <p class="license">
                <a class="icon" rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US"><img alt="Creative Commons License" style="border-width:0" src="http://i.creativecommons.org/l/by-sa/3.0/88x31.png"></a> This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US">Creative Commons Attribution-ShareAlike 3.0 Unported License</a>.
            </p>
        </footer><script async="" defer src="/static/js/common.js">
</script><script id="google-analytics">
    var _gaq = _gaq || [];
            _gaq.push(['_setAccount', 'UA-2097834-1']);
            _gaq.push(['_setDomainName', 'improbable.org']);
            _gaq.push(['_trackPageview']);

            (function() {
                var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
                ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
                var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
            })();
        </script>
    </body>
</html>
