<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title>Early Reports From the ‘Dark Matter’ of the Genome</title>
        <meta name="viewport" content="width=device-width,initial-scale=1.0">
        <link rel="stylesheet" href="/static/css/main.min.css" type="text/css" media="all">
        <!--[if lte IE 8]>
            <link
                rel="stylesheet"
                href="/static/css/ie-fixes.css"
                type="text/css"
                media="all"
            />
            <script src="/static/js/html5shiv.js"></script>
        <![endif]-->

        <meta http-equiv="last-modified" content="Wed, 22 Dec 2010 23:09:27 GMT">
        <link rel="alternate" type="application/atom+xml" title="All Posts (Atom)" href="/feeds/all.atom">
        <link rel="alternate" type="application/rss+xml" title="All Posts (RSS)" href="/feeds/all.rss">
        <link rel="icon" type="image/jpeg" sizes="287x287" href="/static/img/favicon-287x287.jpg">
        <link rel="icon" type="image/jpeg" sizes="128x128" href="/static/img/favicon-128x128.jpg">
    </head>
    <body class="blog post_detail" itemscope itemtype="http://schema.org/BlogPosting">
        <nav id="site-nav">
            <header id="about">
                <h1><a href="/about/">Chris Adams</a></h1>
                <h2>Programmer, cyclist, photographer</h2>
            </header>
            <ul class="links">
                <li>
                    <a href="/">Home</a>
                </li>
                <li>
                    <a href="/about/">About</a>
                </li>
                <li>
                    <a href="/feeds/all.atom" title="Site Feed">Site Feed</a>
                </li>
            </ul>
            <ul class="social-networks">
                <li>
                    <a rel="me" href="https://code4lib.social/@acdha">Mastodon</a>
                </li>
                <li>
                    <a href="https://github.com/acdha" rel="me">Github</a>
                </li>
                <li>
                    <a href="https://bitbucket.org/acdha" rel="me">Bitbucket</a>
                </li>
                <li>
                    <a href="https://pinboard.in/u:acdha/" rel="me">Pinboard</a>
                </li>
                <li>
                    <a href="https://www.flickr.com/photos/acdha" rel="me">Flickr</a>
                </li>
                <li>
                    <a href="https://www.linkedin.com/in/acdha" rel="me">LinkedIn</a>
                </li>
            </ul>
            <div id="site-search"></div>
        </nav>
        <section id="main">
            <article class="post">
                <header>
                    <meta itemprop="dateCreated" content="2010-12-22T18:09:27-04:00">
                    <meta itemprop="dateModified" content="2010-12-22T18:09:27-04:00">
                    <time class="date" itemprop="datePublished" datetime="2010-12-22T22:09:27+00:00">Dec 22</time>
                    <h2 class="" itemprop="title">Early Reports From the ‘Dark Matter’ of the Genome</h2>
                </header>

                <div class="body" itemprop="articleBody"><div class="googlereader description" data-google-id="tag:google.com,2005:reader/item/193b0651fe93dcad">
                        <blockquote>
                            <p>
                                <a href="http://www.wired.com/images_blogs/wiredscience/2010/12/histonelayers.jpg"><img title="histonelayers" src="http://www.wired.com/images_blogs/wiredscience/2010/12/histonelayers.jpg" alt="" width="660" height="356"></a>
                            </p>
                            <p>
                                A collection of new studies on the genomes of two model organisms has moved the frontiers of biology forward, and hints at methods that may someday make real the long-promised, as-yet-unfulfilled genomic revolution.
                            </p>
                            <p>
                                Published in <em>Nature</em> and <em>Science</em>, the studies go far beyond the level of genes that code for proteins, which represent just a small fraction of all genes and an even smaller fraction of all DNA in the genome.
                            </p>
                            <p>
                                Once thought to contain the blueprint of life, protein-coding genes were just the most visible ink in a parts list. The new studies both expand that list and begin to show how the parts are arranged — and how they interact.
                            </p>
                            <p>
                                “It’s become very clear that DNA sequences are just a building block. They don’t explain higher-order complexity,” said Peter Park, a Harvard University bioinformaticist and co-author of one of the <em>Nature</em> studies. “People are sequencing all these genomes, but it doesn’t actually tell us about the activities of the cell.”
                            </p>
                            <p>
                                Park is a contributor to modENCODE, short for the model organism ENCyclopedia of DNA Elements, a massive international collaboration of dozens of institutions and hundreds of researchers. They study an alphabet soup of transcription factors, messengers, regulators and other types of DNA that interact with protein-coding genes to sustain the processes of life.
                            </p>
                            <p>
                                It’s an effort that few people thought necessary a decade ago, when the Human Genome Project’s near-completion was marked by a White House ceremony where President Clinton announced that “it will revolutionize the diagnosis, prevention and treatment of most, if not all, human diseases,” and that “humankind is on the verge of gaining immense, new power to heal.”
                            </p>
                            <p>
                                That may very well still happen, but <a href="http://www.wired.com/wiredscience/2010/03/genome-at-10/">not on the timetable</a> that many scientists expected. With some exceptions, such as breast-cancer-susceptibility mutations and single-gene conditions like Huntington’s disease and Marfan syndrome, identifiable genetic variation has done relatively little to explain disease or development. Promising pathways and mechanisms have been flagged and are now being explored, but understanding is slow in coming.
                            </p>
                            <p>
                                Over the past two years, a series of high-profile genome-wide association tests (comparisons of variation at genomic “hotspots” in thousands of people) suggested new pathways. But they didn’t provide anticipated explanations for disease, and the limitations of standard genomics entered the scientific mainstream. Discussions of “<a href="http://www.wired.com/wiredscience/tag/missing-heritability/">missing heritability</a>,” or the roughly 95 percent of disease risk that’s heritable to the naked eye but can’t be tagged in a sequencer, appeared in the <em>New England Journal of Medicine</em> and <em>Nature</em>.
                            </p>
                            <p>
                                All this represented not a failure, but a dawning realization of just how extraordinarily complicated each genome is. As the process of learning builds on the Human Genome Project’s early steps, researchers are taking fine-grained looks at each genome’s full DNA and chemical components, then trying to understand how all these work together at different scales, from molecules to cells to whole organisms.
                            </p>
                            <p>
                                “The goal of modENCODE is to identify all the functional elements in the genome, and to understand what the genome is doing, which is the next step beyond knowing the sequence,” said Brenton Graveley, a University of Connecticut development biologist.
                            </p>
                            <div style="width:670px">
                                <a href="http://www.wired.com/images_blogs/wiredscience/2010/12/drosophila_layers.jpg"><img title="drosophila_layers" src="http://www.wired.com/images_blogs/wiredscience/2010/12/drosophila_layers.jpg" alt="" width="660" height="253"></a>
                                <p>
                                    An overview of datasets for the fruit fly section of modENCODE. <em>Image:</em> Science
                                </p>
                            </div>
                            <p>
                                In one of the <em>Nature</em> papers, Graveley and dozens of other researchers used new DNA-sequencing techniques to take a base-by-base look at the fruit fly genome, hoping to identify pieces missed in earlier studies. (He compared earlier examinations to “going into a grocery store and not thinking bananas were a fruit, because you the only fruit you know are apples.”)
                            </p>
                            <p>
                                They identified 2,000 previously unknown genes, which now account for one-eighth of the fruit fly’s genome. Beyond that, they identified more than 100,000 new elements, or molecules that aren’t genes but may still have function in the genome. In fruit flies, about 40 percent of the genome fits this description. In humans, it’s closer to two-thirds.
                            </p>
                            <p>
                                The second <em>Nature</em> study looked at non-DNA chemical “information” on the genome, which is made from chromatin: DNA wrapped around proteins called histones, and combined with still more proteins, all of which affect how the DNA works.
                            </p>
                            <p>
                                This approach is known from <a href="http://www.wired.com/medtech/health/news/2005/08/68468">epigenetics</a> (<em>epi</em> means outside) but the new examination was unprecedentedly thorough, looking at dozens of epigenetic factors, at every single DNA base. The resulting “chromatin landscape” revealed regions that once seemed dead, but now appear involved in gene regulation. It’s also just a beginning.
                            </p>
                            <p>
                                “At each location on the sequence, we can measure all these different attributes of chromatin. There are hundreds of attributes, and we only now know what a couple of dozen do,” said Park. “How these marks translate into gene regulation is important. Right now we just see correlations. We don’t necessarily understand the mechanisms behind this.”
                            </p>
                            <div style="width:410px">
                                <a href="http://www.wired.com/images_blogs/wiredscience/2010/12/c_elegans_network2.jpg"><img title="c_elegans_network2" src="http://www.wired.com/images_blogs/wiredscience/2010/12/c_elegans_network2.jpg" alt="" width="400" height="371"></a>
                                <p>
                                    A streamlined representation of the interplay between transcription factors and microRNAs in the roundworm. <em>Image:</em> Science
                                </p>
                            </div>
                            <p>
                                Such mechanisms, and how genetic elements and regulatory layers interact as cells function and organisms develop, is the province of the two <em>Science</em> papers. These provide network-level analyses, or “wiring diagrams,” of the fruit fly and roundworm, said Yale University bioinformaticist Mark Gerstein, co-author of the roundworm paper.
                            </p>
                            <p>
                                Gerstein’s specialty is network structure. In other research he’s compared the characteristics of gene networks between organisms, and even <a href="http://www.wired.com/wiredscience/2010/05/linux-vs-life/">between bacteria and computer operating systems</a>. That work has hinted at the importance of network structure to producing wildly different organisms from common genetic components. (Humans and mice famously share almost the same set of genes.)
                            </p>
                            <p>
                                “Previously, people had looked at transcription factor networks in <em>E. coli</em> and yeast, but nobody had ever looked at this scale of network in an animal,” said Gerstein. “You can start to see patterns: a microRNA that regulates a transcription factor, transcription factor that regulates microRNA, a feedback loop. We observe many of these.”
                            </p>
                            <p>
                                In a commentary accompanying the <em>Science</em> papers, University of Edinburgh geneticist Mark Blaxter likened modENCODE to the Large Hadron Collider, investigating the nature of the genome’s “dark matter.”
                            </p>
                            <p>
                                “It is not currently possible to compute an organism from its genome,” he wrote, but the modENCODE work will “bring this goal closer.”
                            </p>
                            <p>
                                Despite the volume of the studies, joined by <a href="http://genome.cshlp.org/content/early/recent">17 more studies</a> released in tandem in the <em>Journal of Genome Research</em>, the modENCODE work is just beginning. “We’re looking at a vast amount of data. We’re just scratching the surface,” said Park. Future studies will look in greater detail at different tissue types and stages of development.
                            </p>
                            <p>
                                The modENCODE work is also considered a warm-up for a similar project in humans, called ENCODE. It should generate comparable findings in the next two years.
                            </p>
                            <p>
                                “There remains much to be discovered to be discovered even about organisms that are as exhaustively studied as the fruit fly,” said Graveley. “In organisms like humans, there are undoubtedly many, many more mysteries to be uncovered.”
                            </p>
                            <p>
                                <em>Top image: A visualization of physical chromosome arrangement (left) and histone modification readings (right) at a given DNA base location (the green dot).<br>
                                Peter Park/Harvard University.</em>
                            </p>
                            <p>
                                <strong>See Also:</strong>
                            </p>
                            <ul>
                                <li>
                                    <a href="http://www.wired.com/wiredscience/2009/10/beyond-the-genome/">Beyond the Genome</a>
                                </li>
                                <li>
                                    <a href="http://www.wired.com/wiredscience/2010/06/rare-variants/">Rare Gene Glitch a Clue to Genomics Mystery</a>
                                </li>
                                <li>
                                    <a href="http://www.wired.com/wiredscience/2010/06/expanding-genome/">New Form of Gene Regulation Hints at Hidden Dimension of DNA</a>
                                </li>
                                <li>
                                    <a href="http://www.wired.com/wiredscience/2009/04/chromotopo/">To Understand the Blueprint of Life, Crumple It</a>
                                </li>
                                <li>
                                    <a href="http://www.wired.com/wiredscience/2009/11/basics-of-life/">There’s No Such Thing as a ‘Simple’ Organism</a>
                                </li>
                                <li>
                                    <a href="http://www.wired.com/wiredscience/2009/10/fractal-genome/">The Human Genome in 3 Dimensions</a>
                                </li>
                                <li>
                                    <a href="http://www.wired.com/wiredscience/2010/05/linux-vs-life/">Linux vs. Genome in Network Challenge</a>
                                </li>
                                <li>
                                    <a href="http://www.wired.com/wiredscience/2010/03/genome-at-10/">10 Years on, The Genome Revolution Hasn’t Started</a>
                                </li>
                            </ul>
                            <p>
                                <em>Citations: “Integrative Analysis of the Caenorhabditis elegans Genome by the modENCODE Project.” By Mark B. Gerstein, Robert H. Waterston et al.</em> Science<em>, Vol. 330 No. 6012, Dec. 24, 2010</em>
                            </p>
                            <p>
                                <em>“Identification of Functional Elements and Regulatory Circuits by Drosophila modENCODE.” The modENCODE Consortium, Sushmita Roy, Manolis Kelis et al.</em> Science<em>, Vol. 330 No. 6012, Dec. 24, 2010.</em>
                            </p>
                            <p>
                                <em>“Revealing the Dark Matter of the Genome.” By Mark Blaxter.</em> Science<em>, Vol. 330 No. 6012, Dec. 24, 2010.</em>
                            </p>
                            <p>
                                <em>“Comprehensive analysis of the chromatin landscape in</em> Drosophila melanogaster<em>.” By Peter Kharchenko, Peter Park et al.</em> Nature<em>, Vol. 468 No. 7327, Dec. 23, 2010.</em>
                            </p>
                            <p>
                                <em>“The developmental transcriptome of</em> Drosophila melanogaster<em>.” By Brenton Gravely, Susan Celniker et al.</em> Nature<em>, Vol. 468 No. 7327, Dec. 23, 2010.</em>
                            </p>
                        </blockquote>
                    </div>
                    <p class="bookmark-source">
                        Source: <a href="http://www.wired.com/wiredscience/2010/12/genomic-dark-matter/">http://www.wired.com/wiredscience/2010/12/genomic-dark-matter/</a>
                    </p>
                </div>
            </article>

            <nav id="post-nav">


            </nav>
        </section>

        <footer id="site-footer" class="nocontent">
            <p>
                This site is purely my personal work and does not reflect the
                views of my employer.
            </p>
            <p class="license">
                <a class="icon" rel="license" href="https://creativecommons.org/licenses/by-sa/3.0/deed.en_US"><img alt="Creative Commons License" style="border-width: 0" src="https://i.creativecommons.org/l/by-sa/3.0/88x31.png"></a>
                This work is licensed under a
                <a rel="license" href="https://creativecommons.org/licenses/by-sa/3.0/deed.en_US">Creative Commons Attribution-ShareAlike 3.0 Unported
                    License</a>.
            </p>
        </footer>

        <script async src="/static/js/common.js"></script>
        <script async src="https://cdnjs.cloudflare.com/ajax/libs/prettify/r298/run_prettify.min.js" integrity="sha256-hj+5FRlAuvAFANiefn0PpJYCkV1X4QT9EgiPd+6QnCw=" crossorigin="anonymous"></script>
    </body>
</html>
