<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title>When Smaller is Better for GWAS</title>
        <meta name="viewport" content="width=device-width,initial-scale=1.0">
        <link rel="stylesheet" href="/static/css/main.min.css" type="text/css" media="all">
        <!--[if lte IE 8]>
            <link
                rel="stylesheet"
                href="/static/css/ie-fixes.css"
                type="text/css"
                media="all"
            />
            <script src="/static/js/html5shiv.js"></script>
        <![endif]-->

        <meta http-equiv="last-modified" content="Tue, 02 Aug 2011 17:55:05 GMT">
        <link rel="alternate" type="application/atom+xml" title="All Posts (Atom)" href="/feeds/all.atom">
        <link rel="alternate" type="application/rss+xml" title="All Posts (RSS)" href="/feeds/all.rss">
        <link rel="icon" type="image/jpeg" sizes="287x287" href="/static/img/favicon-287x287.jpg">
        <link rel="icon" type="image/jpeg" sizes="128x128" href="/static/img/favicon-128x128.jpg">
    </head>
    <body class="blog post_detail" itemscope itemtype="http://schema.org/BlogPosting">
        <nav id="site-nav">
            <header id="about">
                <h1><a href="/about/">Chris Adams</a></h1>
                <h2>Programmer, cyclist, photographer</h2>
            </header>
            <ul class="links">
                <li>
                    <a href="/">Home</a>
                </li>
                <li>
                    <a href="/about/">About</a>
                </li>
                <li>
                    <a href="/feeds/all.atom" title="Site Feed">Site Feed</a>
                </li>
            </ul>
            <ul class="social-networks">
                <li>
                    <a rel="me" href="https://code4lib.social/@acdha">Mastodon</a>
                </li>
                <li>
                    <a href="https://github.com/acdha" rel="me">Github</a>
                </li>
                <li>
                    <a href="https://bitbucket.org/acdha" rel="me">Bitbucket</a>
                </li>
                <li>
                    <a href="https://pinboard.in/u:acdha/" rel="me">Pinboard</a>
                </li>
                <li>
                    <a href="https://www.flickr.com/photos/acdha" rel="me">Flickr</a>
                </li>
                <li>
                    <a href="https://www.linkedin.com/in/acdha" rel="me">LinkedIn</a>
                </li>
            </ul>
            <div id="site-search"></div>
        </nav>
        <section id="main">
            <article class="post">
                <header>
                    <meta itemprop="dateCreated" content="2011-08-02T13:55:05-04:00">
                    <meta itemprop="dateModified" content="2011-08-02T13:55:05-04:00">
                    <time class="date" itemprop="datePublished" datetime="2011-08-02T17:55:05+00:00">Aug 02</time>
                    <h2 class="" itemprop="title">When Smaller is Better for GWAS</h2>
                </header>

                <div class="body" itemprop="articleBody"><div class="googlereader description" data-google-id="tag:google.com,2005:reader/item/7895e5f5bed07590">
                        <blockquote>
                            <p>
                                <img title="mel-griem-radiation-therapy" src="http://sciencelife.uchospitals.edu/wp-content/uploads/mel-griem-radiation-therapy-243x300.jpg" alt="mel-griem-radiation-therapy" width="243" height="300">
                            </p>
                            <p>
                                <em>By John Easton</em>
                            </p>
                            <p>
                                As of July 2010, nearly 600 genome-wide association studies of 150 distinct diseases and traits had been published. They revealed hundred of specific genomic locations, each with a relatively small effect. There were more than 40 genetic variants, for example, associated just with type 1 diabetes and 30 more related to Crohn’s disease.
                            </p>
                            <p>
                                Despite hundreds of studies, hundreds of thousands of volunteers and billions of genotyped markers, few of the genetic signposts identified “have clear functional implications,” wrote Teri Manolio of the National Human Genome Research Institute in <a href="http://www.nejm.org/doi/full/10.1056/NEJMra0905980">a review article for the <em>New England Journal of Medicine</em></a>. Narrowing an implicated locus to a single variant that directly causes susceptibility to disease by disrupting the expression or function of a protein, he added, “has proved elusive to date.”
                            </p>
                            <p>
                                Enter children’s cancer specialist <a href="http://www.uchospitals.edu/physicians/kenan-onel.html">Kenan Onel</a>, MD, PhD, with a vastly smaller sample size, about 300 - a fraction of the usual GWAS brigade - and a much narrower, tightly focused question. Are there genetic variations, he asked, in patients who were treated with radiation therapy for Hodgkin lymphoma as children and then acquire second cancers decades after treatment?
                            </p>
                            <p>
                                Hodgkin lymphoma is one of the most treatable cancers, with more than 90 percent of patients surviving after a combination of radiation and chemotherapy. But nearly 20 percent of patients treated as children develop a second cancer within 30 years. The younger the patients are when treated and the higher the radiation dose, the greater the risk. This late side effect is the second leading cause of death for long-term Hodgkin’s survivors.
                            </p>
                            <p>
                                In Onel’s GWAS search, <a href="http://www.nature.com/nm/journal/vaop/ncurrent/full/nm.2407.html">published last week in <em>Nature Medicine</em></a>, he found two variants relevant to these secondary cancers. Only three percent of patients with both of the protective versions developed second cancers within 30 years. But those with both of the high-risk variations-a combination found in 50 percent of those of European descent-had <em>ten times</em> the risk: more than 30 percent of them developed second cancers.
                            </p>
                            <p>
                                “This means we can identify children who are most susceptible to radiation-induced cancers before treatment begins and modify their care to prevent this serious long-term complication,” said Onel. “Our options for Hodgkin’s are broad enough that we can find ways to control the initial disease without relying on radiation therapy.”
                            </p>
                            <p>
                                Onel and colleagues “used very wise scientific intuition and they got some place very interesting,” <a href="http://dceg.cancer.gov/about/staff-bios/chanock-stephen">Stephen Channock</a>, head of translational genomics at the National Cancer Institute told <a href="http://blogs.nature.com/nm/spoonful/2011/07/genetic_markers_predict_risk_o.html">Spoonful of Medicine</a>. “It’s a very exciting scientific finding. They did a GWAS in a very small study…but the effect they saw was very strong.”
                            </p>
                            <p>
                                Onel and colleagues analyzed the genomes of 178 Hodgkin’s patients who had been treated between the ages of 8 and 20 with chemotherapy and radiation therapy. Within 30 years after treatment, 96 of them had developed<img title="onel" src="http://sciencelife.uchospitals.edu/wp-content/uploads/onel-199x300.jpg" alt="onel" width="199" height="300"> second cancers and 82 had not.
                            </p>
                            <p>
                                When they scanned each patient’s genome, focusing on 665,313 tiny genetic variations known as SNPs, they found three variations that appeared far more often in patients with second cancers. When they repeated the study using a different set of patients - 62 cases with second cancers and 71 without - two of the three markers were significant.
                            </p>
                            <p>
                                Those two markers were both from a small region known as 21q on chromosome 6. Both are positioned near a gene known as PRDM1. The genetic variations closely associated with increased cancer risk, and with each other, appeared to decrease activation of the PRMD1 gene. They had no detectable effect on any other genes. Cells with the protective version of both markers expressed PRDM1 after being exposed to radiation. Cells with the variants linked to subsequent cancers did not produce any PRDM1.
                            </p>
                            <p>
                                Previous studies have found that PRDM1 is involved in a variety of fundamental cellular processes, including proliferation, differentiation and apoptosis - which can all go awry in cancer. The gene’s activity is lost in many cancer types.
                            </p>
                            <p>
                                “Taken together,” the authors note, “our findings support a novel role for PRDM1 as a radiation-responsive tumor suppressor.” PRMD1 may be important for understanding the causes of second cancers in survivors of pediatric Hodgkin’s lymphoma as well as in other cancer patients treated with radiation therapy.”
                            </p>
                            <p>
                                This study should also “bring some optimism” back to genome-wide association studies, Onel added.
                            </p>
                            <p>
                                Most previous cancer-related markers found through GWAS have been “of little clinical value for predicting risk, response to therapy or survival.” But by incorporating environmental exposure, such as radiation therapy, into genomic investigations, “much of the missing heritability can be revealed,” he said.
                            </p>
                            <p>
                                “The idea is that different exposures elicit different cellular responses, each with its own set of genetic determinants. Thus, by not accounting for them in GWASs,”, “you are by definition going to obscure and attenuate disease-related genetic signals.”
                            </p>
                            <p>
                                So the second take-home message is that “context matters,” Onel told <a href="http://www.medscape.com/viewarticle/747006">Medscape Medical News</a>. “Our SNPs are very common in individuals of European descent, but they only contribute to cancer risk in the context of radiation exposure… In the absence of exposure information, they will appear neutral in genetic studies and not be identified.”
                            </p>
                            <p>
                                =====
                            </p>
                            <p>
                                <span title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.jtitle=Nature+Medicine&amp;rft_id=info%3Adoi%2F10.1038%2Fnm.2407&amp;rfr_id=info%3Asid%2Fresearchblogging.org&amp;rft.atitle=Variants+at+6q21+implicate+PRDM1+in+the+etiology+of+therapy-induced+second+malignancies+after+Hodgkin%27s+lymphoma&amp;rft.issn=1078-8956&amp;rft.date=2011&amp;rft.volume=&amp;rft.issue=&amp;rft.spage=&amp;rft.epage=&amp;rft.artnum=http%3A%2F%2Fwww.nature.com%2Fdoifinder%2F10.1038%2Fnm.2407&amp;rft.au=Best%2C+T.&amp;rft.au=Li%2C+D.&amp;rft.au=Skol%2C+A.&amp;rft.au=Kirchhoff%2C+T.&amp;rft.au=Jackson%2C+S.&amp;rft.au=Yasui%2C+Y.&amp;rft.au=Bhatia%2C+S.&amp;rft.au=Strong%2C+L.&amp;rft.au=Domchek%2C+S.&amp;rft.au=Nathanson%2C+K.&amp;rft.au=Olopade%2C+O.&amp;rft.au=Huang%2C+R.&amp;rft.au=Mack%2C+T.&amp;rft.au=Conti%2C+D.&amp;rft.au=Offit%2C+K.&amp;rft.au=Cozen%2C+W.&amp;rft.au=Robison%2C+L.&amp;rft.au=Onel%2C+K.&amp;rfe_dat=bpr3.included=1;bpr3.tags=Biology%2CMedicine%2CGenetics%2C+Cancer">Best, T., Li, D., Skol, A., Kirchhoff, T., Jackson, S., Yasui, Y., Bhatia, S., Strong, L., Domchek, S., Nathanson, K., Olopade, O., Huang, R., Mack, T., Conti, D., Offit, K., Cozen, W., Robison, L., &amp; Onel, K. (2011). Variants at 6q21 implicate PRDM1 in the etiology of therapy-induced second malignancies after Hodgkin’s lymphoma <span style="font-style:italic">Nature Medicine</span> DOI: <a href="http://dx.doi.org/10.1038/nm.2407">10.1038/nm.2407</a></span>
                            </p>
                        </blockquote>
                    </div>
                    <p class="bookmark-source">
                        Source: <a href="http://sciencelife.uchospitals.edu/2011/08/02/when-smaller-is-better-for-gwas/">http://sciencelife.uchospitals.edu/2011/08/02/when-smaller-is-better-for-gwas/</a>
                    </p>
                </div>
            </article>

            <nav id="post-nav">


            </nav>
        </section>

        <footer id="site-footer" class="nocontent">
            <p>
                This site is purely my personal work and does not reflect the
                views of my employer.
            </p>
            <p class="license">
                <a class="icon" rel="license" href="https://creativecommons.org/licenses/by-sa/3.0/deed.en_US"><img alt="Creative Commons License" style="border-width: 0" src="https://i.creativecommons.org/l/by-sa/3.0/88x31.png"></a>
                This work is licensed under a
                <a rel="license" href="https://creativecommons.org/licenses/by-sa/3.0/deed.en_US">Creative Commons Attribution-ShareAlike 3.0 Unported
                    License</a>.
            </p>
        </footer>

        <script async src="/static/js/common.js"></script>
        <script async src="https://cdnjs.cloudflare.com/ajax/libs/prettify/r298/run_prettify.min.js" integrity="sha256-hj+5FRlAuvAFANiefn0PpJYCkV1X4QT9EgiPd+6QnCw=" crossorigin="anonymous"></script>
    </body>
</html>
